.changeset/brave-snails-camp.md

```diff
@@ -0,0 +1,19 @@
+---
+'merchant-center-application-template-starter': patch
+'@commercetools-backend/express': patch
+'@commercetools-backend/loggers': patch
+'@commercetools-frontend/actions-global': patch
+'@commercetools-frontend/application-components': patch
+'@commercetools-frontend/application-shell': patch
+'@commercetools-frontend/l10n': patch
+'@commercetools-frontend/mc-scripts': patch
+'@commercetools-frontend/react-notifications': patch
+'@commercetools-frontend/sdk': patch
+'@commercetools-frontend/sentry': patch
+'playground': patch
+'@commercetools-local/visual-testing-app': patch
+'@commercetools-website/custom-applications': patch
+'@commercetools-website/components-playground': patch
+---
+
+Update dependencies and regenerate l10n data based on CLDR v37
```

Comment:
FYI: regenerating the l10n data yielded this output

<details>
<summary>Generate l10n data output</summary>

```
❯ yarn generate-data                                                     
yarn run v1.22.4
$ node ./scripts/generate-l10n-data.js
[country] Writing en data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/countries/en.json
[country] Writing de data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/countries/de.json
[country] Writing es data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/countries/es.json
[country] Writing fr-FR data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/countries/fr-FR.json
[country] Writing zh-CN data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/countries/zh-CN.json
[country] Writing ja data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/countries/ja.json
[timezone] Writing en data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/time-zones/en.json
[timezone] Writing de data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/time-zones/de.json
[timezone] Writing es data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/time-zones/es.json
[timezone] Writing fr-FR data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/time-zones/fr-FR.json
[timezone] Writing zh-CN data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/time-zones/zh-CN.json
[timezone] Writing ja data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/time-zones/ja.json
[language] Writing en data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/languages/en.json
[language] Writing de data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/languages/de.json
[language] Writing es data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/languages/es.json
[language] Writing fr-FR data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/languages/fr-FR.json
[language] Writing zh-CN data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/languages/zh-CN.json
[language] Writing ja data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/languages/ja.json
[country] Updated 6 locales


[country] Data generated with warnings for locale "de".

The field "sz" has changed from "Swasiland" to "Eswatini".


[country] Data generated with warnings for locale "es".

The field "mk" has changed from "Macedonia" to "Macedonia del Norte".
The field "xa" has been added ('"Pseudoacentos"').
The field "xb" has been added ('"Pseudobidi"').


[country] Data generated with warnings for locale "fr-FR".

The field "mk" has changed from "Macédoine" to "Macédoine du Nord".


[country] Data generated with warnings for locale "ja".

The field "xa" has been added ('"疑似アクセント"').
The field "xb" has been added ('"疑似 BIDI"').
[timezone] Updated 6 locales


[timezone] Data generated with warnings for locale "en".

The field "America/Adak.abbr" has changed from "HST" to "HDT".
The field "America/Adak.offset" has changed from "-10:00" to "-09:00".
The field "America/Atka.abbr" has changed from "HST" to "HDT".
The field "America/Atka.offset" has changed from "-10:00" to "-09:00".
The field "US/Aleutian.abbr" has changed from "HST" to "HDT".
The field "US/Aleutian.offset" has changed from "-10:00" to "-09:00".
The field "America/Anchorage.abbr" has changed from "AKST" to "AKDT".
The field "America/Anchorage.offset" has changed from "-09:00" to "-08:00".
The field "America/Juneau.abbr" has changed from "AKST" to "AKDT".
The field "America/Juneau.offset" has changed from "-09:00" to "-08:00".
The field "America/Metlakatla.abbr" has changed from "AKST" to "AKDT".
The field "America/Metlakatla.offset" has changed from "-09:00" to "-08:00".
The field "America/Nome.abbr" has changed from "AKST" to "AKDT".
The field "America/Nome.offset" has changed from "-09:00" to "-08:00".
The field "America/Sitka.abbr" has changed from "AKST" to "AKDT".
The field "America/Sitka.offset" has changed from "-09:00" to "-08:00".
The field "America/Yakutat.abbr" has changed from "AKST" to "AKDT".
The field "America/Yakutat.offset" has changed from "-09:00" to "-08:00".
The field "US/Alaska.abbr" has changed from "AKST" to "AKDT".
The field "US/Alaska.offset" has changed from "-09:00" to "-08:00".
The field "America/Dawson.abbr" has changed from "PST" to "MST".
The field "America/Dawson.offset" has changed from "-08:00" to "-07:00".
The field "America/Ensenada.abbr" has changed from "PST" to "PDT".
The field "America/Ensenada.offset" has changed from "-08:00" to "-07:00".
The field "America/Los_Angeles.abbr" has changed from "PST" to "PDT".
The field "America/Los_Angeles.offset" has changed from "-08:00" to "-07:00".
The field "America/Santa_Isabel.abbr" has changed from "PST" to "PDT".
The field "America/Santa_Isabel.offset" has changed from "-08:00" to "-07:00".
The field "America/Tijuana.abbr" has changed from "PST" to "PDT".
The field "America/Tijuana.offset" has changed from "-08:00" to "-07:00".
The field "America/Vancouver.abbr" has changed from "PST" to "PDT".
The field "America/Vancouver.offset" has changed from "-08:00" to "-07:00".
The field "America/Whitehorse.abbr" has changed from "PST" to "MST".
The field "America/Whitehorse.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Pacific.abbr" has changed from "PST" to "PDT".
The field "Canada/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Yukon.abbr" has changed from "PST" to "MST".
The field "Canada/Yukon.offset" has changed from "-08:00" to "-07:00".
The field "Mexico/BajaNorte.abbr" has changed from "PST" to "PDT".
The field "Mexico/BajaNorte.offset" has changed from "-08:00" to "-07:00".
The field "PST8PDT.abbr" has changed from "PST" to "PDT".
The field "PST8PDT.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific.abbr" has changed from "PST" to "PDT".
The field "US/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific-New.abbr" has changed from "PST" to "PDT".
The field "US/Pacific-New.offset" has changed from "-08:00" to "-07:00".
The field "America/Boise.abbr" has changed from "MST" to "MDT".
The field "America/Boise.offset" has changed from "-07:00" to "-06:00".
The field "America/Cambridge_Bay.abbr" has changed from "MST" to "MDT".
The field "America/Cambridge_Bay.offset" has changed from "-07:00" to "-06:00".
The field "America/Chihuahua.abbr" has changed from "MST" to "MDT".
The field "America/Chihuahua.offset" has changed from "-07:00" to "-06:00".
The field "America/Denver.abbr" has changed from "MST" to "MDT".
The field "America/Denver.offset" has changed from "-07:00" to "-06:00".
The field "America/Edmonton.abbr" has changed from "MST" to "MDT".
The field "America/Edmonton.offset" has changed from "-07:00" to "-06:00".
The field "America/Inuvik.abbr" has changed from "MST" to "MDT".
The field "America/Inuvik.offset" has changed from "-07:00" to "-06:00".
The field "America/Mazatlan.abbr" has changed from "MST" to "MDT".
The field "America/Mazatlan.offset" has changed from "-07:00" to "-06:00".
The field "America/Ojinaga.abbr" has changed from "MST" to "MDT".
The field "America/Ojinaga.offset" has changed from "-07:00" to "-06:00".
The field "America/Shiprock.abbr" has changed from "MST" to "MDT".
The field "America/Shiprock.offset" has changed from "-07:00" to "-06:00".
The field "America/Yellowknife.abbr" has changed from "MST" to "MDT".
The field "America/Yellowknife.offset" has changed from "-07:00" to "-06:00".
The field "Canada/Mountain.abbr" has changed from "MST" to "MDT".
The field "Canada/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "MST7MDT.abbr" has changed from "MST" to "MDT".
The field "MST7MDT.offset" has changed from "-07:00" to "-06:00".
The field "Mexico/BajaSur.abbr" has changed from "MST" to "MDT".
The field "Mexico/BajaSur.offset" has changed from "-07:00" to "-06:00".
The field "Navajo.abbr" has changed from "MST" to "MDT".
The field "Navajo.offset" has changed from "-07:00" to "-06:00".
The field "US/Mountain.abbr" has changed from "MST" to "MDT".
The field "US/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "America/Bahia_Banderas.abbr" has changed from "CST" to "CDT".
The field "America/Bahia_Banderas.offset" has changed from "-06:00" to "-05:00".
The field "America/Chicago.abbr" has changed from "CST" to "CDT".
The field "America/Chicago.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Knox.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Knox.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Tell_City.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Tell_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Knox_IN.abbr" has changed from "CST" to "CDT".
The field "America/Knox_IN.offset" has changed from "-06:00" to "-05:00".
The field "America/Matamoros.abbr" has changed from "CST" to "CDT".
The field "America/Matamoros.offset" has changed from "-06:00" to "-05:00".
The field "America/Menominee.abbr" has changed from "CST" to "CDT".
The field "America/Menominee.offset" has changed from "-06:00" to "-05:00".
The field "America/Merida.abbr" has changed from "CST" to "CDT".
The field "America/Merida.offset" has changed from "-06:00" to "-05:00".
The field "America/Mexico_City.abbr" has changed from "CST" to "CDT".
The field "America/Mexico_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Monterrey.abbr" has changed from "CST" to "CDT".
The field "America/Monterrey.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Beulah.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Beulah.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Center.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Center.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/New_Salem.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/New_Salem.offset" has changed from "-06:00" to "-05:00".
The field "America/Rainy_River.abbr" has changed from "CST" to "CDT".
The field "America/Rainy_River.offset" has changed from "-06:00" to "-05:00".
The field "America/Rankin_Inlet.abbr" has changed from "CST" to "CDT".
The field "America/Rankin_Inlet.offset" has changed from "-06:00" to "-05:00".
The field "America/Resolute.abbr" has changed from "CST" to "CDT".
The field "America/Resolute.offset" has changed from "-06:00" to "-05:00".
The field "America/Winnipeg.abbr" has changed from "CST" to "CDT".
The field "America/Winnipeg.offset" has changed from "-06:00" to "-05:00".
The field "CST6CDT.abbr" has changed from "CST" to "CDT".
The field "CST6CDT.offset" has changed from "-06:00" to "-05:00".
The field "Canada/Central.abbr" has changed from "CST" to "CDT".
The field "Canada/Central.offset" has changed from "-06:00" to "-05:00".
The field "Mexico/General.abbr" has changed from "CST" to "CDT".
The field "Mexico/General.offset" has changed from "-06:00" to "-05:00".
The field "US/Central.abbr" has changed from "CST" to "CDT".
The field "US/Central.offset" has changed from "-06:00" to "-05:00".
The field "US/Indiana-Starke.abbr" has changed from "CST" to "CDT".
The field "US/Indiana-Starke.offset" has changed from "-06:00" to "-05:00".
The field "America/Detroit.abbr" has changed from "EST" to "EDT".
The field "America/Detroit.offset" has changed from "-05:00" to "-04:00".
The field "America/Fort_Wayne.abbr" has changed from "EST" to "EDT".
The field "America/Fort_Wayne.offset" has changed from "-05:00" to "-04:00".
The field "America/Grand_Turk.abbr" has changed from "EST" to "EDT".
The field "America/Grand_Turk.offset" has changed from "-05:00" to "-04:00".
The field "America/Havana.abbr" has changed from "CST" to "CDT".
The field "America/Havana.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Marengo.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Marengo.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Petersburg.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Petersburg.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vevay.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vevay.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vincennes.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vincennes.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Winamac.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Winamac.offset" has changed from "-05:00" to "-04:00".
The field "America/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Iqaluit.abbr" has changed from "EST" to "EDT".
The field "America/Iqaluit.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Monticello.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Monticello.offset" has changed from "-05:00" to "-04:00".
The field "America/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Montreal.abbr" has changed from "EST" to "EDT".
The field "America/Montreal.offset" has changed from "-05:00" to "-04:00".
The field "America/Nassau.abbr" has changed from "EST" to "EDT".
The field "America/Nassau.offset" has changed from "-05:00" to "-04:00".
The field "America/New_York.abbr" has changed from "EST" to "EDT".
The field "America/New_York.offset" has changed from "-05:00" to "-04:00".
The field "America/Nipigon.abbr" has changed from "EST" to "EDT".
The field "America/Nipigon.offset" has changed from "-05:00" to "-04:00".
The field "America/Pangnirtung.abbr" has changed from "EST" to "EDT".
The field "America/Pangnirtung.offset" has changed from "-05:00" to "-04:00".
The field "America/Port-au-Prince.abbr" has changed from "EST" to "EDT".
The field "America/Port-au-Prince.offset" has changed from "-05:00" to "-04:00".
The field "America/Thunder_Bay.abbr" has changed from "EST" to "EDT".
The field "America/Thunder_Bay.offset" has changed from "-05:00" to "-04:00".
The field "America/Toronto.abbr" has changed from "EST" to "EDT".
The field "America/Toronto.offset" has changed from "-05:00" to "-04:00".
The field "Canada/Eastern.abbr" has changed from "EST" to "EDT".
The field "Canada/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "Chile/EasterIsland.abbr" has changed from "-05" to "-06".
The field "Chile/EasterIsland.offset" has changed from "-05:00" to "-06:00".
The field "Cuba.abbr" has changed from "CST" to "CDT".
The field "Cuba.offset" has changed from "-05:00" to "-04:00".
The field "EST5EDT.abbr" has changed from "EST" to "EDT".
The field "EST5EDT.offset" has changed from "-05:00" to "-04:00".
The field "Pacific/Easter.abbr" has changed from "-05" to "-06".
The field "Pacific/Easter.offset" has changed from "-05:00" to "-06:00".
The field "US/East-Indiana.abbr" has changed from "EST" to "EDT".
The field "US/East-Indiana.offset" has changed from "-05:00" to "-04:00".
The field "US/Eastern.abbr" has changed from "EST" to "EDT".
The field "US/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "US/Michigan.abbr" has changed from "EST" to "EDT".
The field "US/Michigan.offset" has changed from "-05:00" to "-04:00".
The field "America/Glace_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Glace_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Goose_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Goose_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Halifax.abbr" has changed from "AST" to "ADT".
The field "America/Halifax.offset" has changed from "-04:00" to "-03:00".
The field "America/Moncton.abbr" has changed from "AST" to "ADT".
The field "America/Moncton.offset" has changed from "-04:00" to "-03:00".
The field "America/Thule.abbr" has changed from "AST" to "ADT".
The field "America/Thule.offset" has changed from "-04:00" to "-03:00".
The field "Atlantic/Bermuda.abbr" has changed from "AST" to "ADT".
The field "Atlantic/Bermuda.offset" has changed from "-04:00" to "-03:00".
The field "Canada/Atlantic.abbr" has changed from "AST" to "ADT".
The field "Canada/Atlantic.offset" has changed from "-04:00" to "-03:00".
The field "America/St_Johns.abbr" has changed from "NST" to "NDT".
The field "America/St_Johns.offset" has changed from "-03:30" to "-02:30".
The field "Canada/Newfoundland.abbr" has changed from "NST" to "NDT".
The field "Canada/Newfoundland.offset" has changed from "-03:30" to "-02:30".
The field "America/Asuncion.abbr" has changed from "-03" to "-04".
The field "America/Asuncion.offset" has changed from "-03:00" to "-04:00".
The field "America/Godthab.abbr" has changed from "-03" to "-02".
The field "America/Godthab.offset" has changed from "-03:00" to "-02:00".
The field "America/Miquelon.abbr" has changed from "-03" to "-02".
The field "America/Miquelon.offset" has changed from "-03:00" to "-02:00".
The field "America/Santiago.abbr" has changed from "-03" to "-04".
The field "America/Santiago.offset" has changed from "-03:00" to "-04:00".
The field "Chile/Continental.abbr" has changed from "-03" to "-04".
The field "Chile/Continental.offset" has changed from "-03:00" to "-04:00".
The field "America/Scoresbysund.abbr" has changed from "-01" to "+00".
The field "America/Scoresbysund.offset" has changed from "-01:00" to "+00:00".
The field "Atlantic/Azores.abbr" has changed from "-01" to "+00".
The field "Atlantic/Azores.offset" has changed from "-01:00" to "+00:00".
The field "Antarctica/Troll.abbr" has changed from "+00" to "+02".
The field "Antarctica/Troll.offset" has changed from "+00:00" to "+02:00".
The field "Atlantic/Canary.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Canary.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faeroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faeroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Madeira.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Madeira.offset" has changed from "+00:00" to "+01:00".
The field "Eire.abbr" has changed from "GMT" to "IST".
The field "Eire.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Belfast.abbr" has changed from "GMT" to "BST".
The field "Europe/Belfast.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Dublin.abbr" has changed from "GMT" to "IST".
The field "Europe/Dublin.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Guernsey.abbr" has changed from "GMT" to "BST".
The field "Europe/Guernsey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Isle_of_Man.abbr" has changed from "GMT" to "BST".
The field "Europe/Isle_of_Man.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Jersey.abbr" has changed from "GMT" to "BST".
The field "Europe/Jersey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Lisbon.abbr" has changed from "WET" to "WEST".
The field "Europe/Lisbon.offset" has changed from "+00:00" to "+01:00".
The field "Europe/London.abbr" has changed from "GMT" to "BST".
The field "Europe/London.offset" has changed from "+00:00" to "+01:00".
The field "GB.abbr" has changed from "GMT" to "BST".
The field "GB.offset" has changed from "+00:00" to "+01:00".
The field "GB-Eire.abbr" has changed from "GMT" to "BST".
The field "GB-Eire.offset" has changed from "+00:00" to "+01:00".
The field "Portugal.abbr" has changed from "WET" to "WEST".
The field "Portugal.offset" has changed from "+00:00" to "+01:00".
The field "WET.abbr" has changed from "WET" to "WEST".
The field "WET.offset" has changed from "+00:00" to "+01:00".
The field "Africa/Ceuta.abbr" has changed from "CET" to "CEST".
The field "Africa/Ceuta.offset" has changed from "+01:00" to "+02:00".
The field "Arctic/Longyearbyen.abbr" has changed from "CET" to "CEST".
The field "Arctic/Longyearbyen.offset" has changed from "+01:00" to "+02:00".
The field "Atlantic/Jan_Mayen.abbr" has changed from "CET" to "CEST".
The field "Atlantic/Jan_Mayen.offset" has changed from "+01:00" to "+02:00".
The field "CET.abbr" has changed from "CET" to "CEST".
The field "CET.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Amsterdam.abbr" has changed from "CET" to "CEST".
The field "Europe/Amsterdam.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Andorra.abbr" has changed from "CET" to "CEST".
The field "Europe/Andorra.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Belgrade.abbr" has changed from "CET" to "CEST".
The field "Europe/Belgrade.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Berlin.abbr" has changed from "CET" to "CEST".
The field "Europe/Berlin.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Bratislava.abbr" has changed from "CET" to "CEST".
The field "Europe/Bratislava.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Brussels.abbr" has changed from "CET" to "CEST".
The field "Europe/Brussels.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Budapest.abbr" has changed from "CET" to "CEST".
The field "Europe/Budapest.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Busingen.abbr" has changed from "CET" to "CEST".
The field "Europe/Busingen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Copenhagen.abbr" has changed from "CET" to "CEST".
The field "Europe/Copenhagen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Gibraltar.abbr" has changed from "CET" to "CEST".
The field "Europe/Gibraltar.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Ljubljana.abbr" has changed from "CET" to "CEST".
The field "Europe/Ljubljana.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Luxembourg.abbr" has changed from "CET" to "CEST".
The field "Europe/Luxembourg.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Madrid.abbr" has changed from "CET" to "CEST".
The field "Europe/Madrid.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Malta.abbr" has changed from "CET" to "CEST".
The field "Europe/Malta.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Monaco.abbr" has changed from "CET" to "CEST".
The field "Europe/Monaco.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Oslo.abbr" has changed from "CET" to "CEST".
The field "Europe/Oslo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Paris.abbr" has changed from "CET" to "CEST".
The field "Europe/Paris.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Podgorica.abbr" has changed from "CET" to "CEST".
The field "Europe/Podgorica.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Prague.abbr" has changed from "CET" to "CEST".
The field "Europe/Prague.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Rome.abbr" has changed from "CET" to "CEST".
The field "Europe/Rome.offset" has changed from "+01:00" to "+02:00".
The field "Europe/San_Marino.abbr" has changed from "CET" to "CEST".
The field "Europe/San_Marino.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Sarajevo.abbr" has changed from "CET" to "CEST".
The field "Europe/Sarajevo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Skopje.abbr" has changed from "CET" to "CEST".
The field "Europe/Skopje.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Stockholm.abbr" has changed from "CET" to "CEST".
The field "Europe/Stockholm.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Tirane.abbr" has changed from "CET" to "CEST".
The field "Europe/Tirane.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vaduz.abbr" has changed from "CET" to "CEST".
The field "Europe/Vaduz.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vatican.abbr" has changed from "CET" to "CEST".
The field "Europe/Vatican.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vienna.abbr" has changed from "CET" to "CEST".
The field "Europe/Vienna.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Warsaw.abbr" has changed from "CET" to "CEST".
The field "Europe/Warsaw.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zagreb.abbr" has changed from "CET" to "CEST".
The field "Europe/Zagreb.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zurich.abbr" has changed from "CET" to "CEST".
The field "Europe/Zurich.offset" has changed from "+01:00" to "+02:00".
The field "MET.abbr" has changed from "MET" to "MEST".
The field "MET.offset" has changed from "+01:00" to "+02:00".
The field "Poland.abbr" has changed from "CET" to "CEST".
The field "Poland.offset" has changed from "+01:00" to "+02:00".
The field "Asia/Amman.abbr" has changed from "EET" to "EEST".
The field "Asia/Amman.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Beirut.abbr" has changed from "EET" to "EEST".
The field "Asia/Beirut.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Damascus.abbr" has changed from "EET" to "EEST".
The field "Asia/Damascus.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Famagusta.abbr" has changed from "EET" to "EEST".
The field "Asia/Famagusta.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Gaza.abbr" has changed from "EET" to "EEST".
The field "Asia/Gaza.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Hebron.abbr" has changed from "EET" to "EEST".
The field "Asia/Hebron.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Jerusalem.abbr" has changed from "IST" to "IDT".
The field "Asia/Jerusalem.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Asia/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tel_Aviv.abbr" has changed from "IST" to "IDT".
The field "Asia/Tel_Aviv.offset" has changed from "+02:00" to "+03:00".
The field "EET.abbr" has changed from "EET" to "EEST".
The field "EET.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Athens.abbr" has changed from "EET" to "EEST".
The field "Europe/Athens.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Bucharest.abbr" has changed from "EET" to "EEST".
The field "Europe/Bucharest.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Chisinau.abbr" has changed from "EET" to "EEST".
The field "Europe/Chisinau.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Helsinki.abbr" has changed from "EET" to "EEST".
The field "Europe/Helsinki.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Kiev.abbr" has changed from "EET" to "EEST".
The field "Europe/Kiev.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Mariehamn.abbr" has changed from "EET" to "EEST".
The field "Europe/Mariehamn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Europe/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Riga.abbr" has changed from "EET" to "EEST".
The field "Europe/Riga.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Sofia.abbr" has changed from "EET" to "EEST".
The field "Europe/Sofia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tallinn.abbr" has changed from "EET" to "EEST".
The field "Europe/Tallinn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tiraspol.abbr" has changed from "EET" to "EEST".
The field "Europe/Tiraspol.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Uzhgorod.abbr" has changed from "EET" to "EEST".
The field "Europe/Uzhgorod.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Vilnius.abbr" has changed from "EET" to "EEST".
The field "Europe/Vilnius.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Zaporozhye.abbr" has changed from "EET" to "EEST".
The field "Europe/Zaporozhye.offset" has changed from "+02:00" to "+03:00".
The field "Israel.abbr" has changed from "IST" to "IDT".
The field "Israel.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tehran.abbr" has changed from "+0330" to "+0430".
The field "Asia/Tehran.offset" has changed from "+03:30" to "+04:30".
The field "Iran.abbr" has changed from "+0330" to "+0430".
The field "Iran.offset" has changed from "+03:30" to "+04:30".
The field "Australia/Adelaide.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Adelaide.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Broken_Hill.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Broken_Hill.offset" has changed from "+10:30" to "+09:30".
The field "Australia/South.abbr" has changed from "ACDT" to "ACST".
The field "Australia/South.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Yancowinna.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Yancowinna.offset" has changed from "+10:30" to "+09:30".
The field "Australia/ACT.abbr" has changed from "AEDT" to "AEST".
The field "Australia/ACT.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Canberra.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Canberra.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Currie.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Currie.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Hobart.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Hobart.offset" has changed from "+11:00" to "+10:00".
The field "Australia/LHI.abbr" has changed from "+11" to "+1030".
The field "Australia/LHI.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Lord_Howe.abbr" has changed from "+11" to "+1030".
The field "Australia/Lord_Howe.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Melbourne.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Melbourne.offset" has changed from "+11:00" to "+10:00".
The field "Australia/NSW.abbr" has changed from "AEDT" to "AEST".
The field "Australia/NSW.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Sydney.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Sydney.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Tasmania.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Tasmania.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Victoria.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Victoria.offset" has changed from "+11:00" to "+10:00".
The field "Pacific/Norfolk.abbr" has changed from "+12" to "+11".
The field "Pacific/Norfolk.offset" has changed from "+12:00" to "+11:00".
The field "Antarctica/McMurdo.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/McMurdo.offset" has changed from "+13:00" to "+12:00".
The field "Antarctica/South_Pole.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/South_Pole.offset" has changed from "+13:00" to "+12:00".
The field "NZ.abbr" has changed from "NZDT" to "NZST".
The field "NZ.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Auckland.abbr" has changed from "NZDT" to "NZST".
The field "Pacific/Auckland.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Fiji.abbr" has changed from "+13" to "+12".
The field "Pacific/Fiji.offset" has changed from "+13:00" to "+12:00".
The field "NZ-CHAT.abbr" has changed from "+1345" to "+1245".
The field "NZ-CHAT.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Chatham.abbr" has changed from "+1345" to "+1245".
The field "Pacific/Chatham.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Apia.abbr" has changed from "+14" to "+13".
The field "Pacific/Apia.offset" has changed from "+14:00" to "+13:00".
The field "America/Nuuk" has been added ('{"name":"America/Nuuk","abbr":"-02","offset":"-02:00"}').


[timezone] Data generated with warnings for locale "de".

The field "America/Adak.abbr" has changed from "HST" to "HDT".
The field "America/Adak.offset" has changed from "-10:00" to "-09:00".
The field "America/Atka.abbr" has changed from "HST" to "HDT".
The field "America/Atka.offset" has changed from "-10:00" to "-09:00".
The field "US/Aleutian.abbr" has changed from "HST" to "HDT".
The field "US/Aleutian.offset" has changed from "-10:00" to "-09:00".
The field "America/Anchorage.abbr" has changed from "AKST" to "AKDT".
The field "America/Anchorage.offset" has changed from "-09:00" to "-08:00".
The field "America/Juneau.abbr" has changed from "AKST" to "AKDT".
The field "America/Juneau.offset" has changed from "-09:00" to "-08:00".
The field "America/Metlakatla.abbr" has changed from "AKST" to "AKDT".
The field "America/Metlakatla.offset" has changed from "-09:00" to "-08:00".
The field "America/Nome.abbr" has changed from "AKST" to "AKDT".
The field "America/Nome.offset" has changed from "-09:00" to "-08:00".
The field "America/Sitka.abbr" has changed from "AKST" to "AKDT".
The field "America/Sitka.offset" has changed from "-09:00" to "-08:00".
The field "America/Yakutat.abbr" has changed from "AKST" to "AKDT".
The field "America/Yakutat.offset" has changed from "-09:00" to "-08:00".
The field "US/Alaska.abbr" has changed from "AKST" to "AKDT".
The field "US/Alaska.offset" has changed from "-09:00" to "-08:00".
The field "America/Dawson.abbr" has changed from "PST" to "MST".
The field "America/Dawson.offset" has changed from "-08:00" to "-07:00".
The field "America/Ensenada.abbr" has changed from "PST" to "PDT".
The field "America/Ensenada.offset" has changed from "-08:00" to "-07:00".
The field "America/Los_Angeles.abbr" has changed from "PST" to "PDT".
The field "America/Los_Angeles.offset" has changed from "-08:00" to "-07:00".
The field "America/Santa_Isabel.abbr" has changed from "PST" to "PDT".
The field "America/Santa_Isabel.offset" has changed from "-08:00" to "-07:00".
The field "America/Tijuana.abbr" has changed from "PST" to "PDT".
The field "America/Tijuana.offset" has changed from "-08:00" to "-07:00".
The field "America/Vancouver.abbr" has changed from "PST" to "PDT".
The field "America/Vancouver.offset" has changed from "-08:00" to "-07:00".
The field "America/Whitehorse.abbr" has changed from "PST" to "MST".
The field "America/Whitehorse.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Pacific.abbr" has changed from "PST" to "PDT".
The field "Canada/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Yukon.abbr" has changed from "PST" to "MST".
The field "Canada/Yukon.offset" has changed from "-08:00" to "-07:00".
The field "Mexico/BajaNorte.abbr" has changed from "PST" to "PDT".
The field "Mexico/BajaNorte.offset" has changed from "-08:00" to "-07:00".
The field "PST8PDT.abbr" has changed from "PST" to "PDT".
The field "PST8PDT.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific.abbr" has changed from "PST" to "PDT".
The field "US/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific-New.abbr" has changed from "PST" to "PDT".
The field "US/Pacific-New.offset" has changed from "-08:00" to "-07:00".
The field "America/Boise.abbr" has changed from "MST" to "MDT".
The field "America/Boise.offset" has changed from "-07:00" to "-06:00".
The field "America/Cambridge_Bay.abbr" has changed from "MST" to "MDT".
The field "America/Cambridge_Bay.offset" has changed from "-07:00" to "-06:00".
The field "America/Chihuahua.abbr" has changed from "MST" to "MDT".
The field "America/Chihuahua.offset" has changed from "-07:00" to "-06:00".
The field "America/Denver.abbr" has changed from "MST" to "MDT".
The field "America/Denver.offset" has changed from "-07:00" to "-06:00".
The field "America/Edmonton.abbr" has changed from "MST" to "MDT".
The field "America/Edmonton.offset" has changed from "-07:00" to "-06:00".
The field "America/Inuvik.abbr" has changed from "MST" to "MDT".
The field "America/Inuvik.offset" has changed from "-07:00" to "-06:00".
The field "America/Mazatlan.abbr" has changed from "MST" to "MDT".
The field "America/Mazatlan.offset" has changed from "-07:00" to "-06:00".
The field "America/Ojinaga.abbr" has changed from "MST" to "MDT".
The field "America/Ojinaga.offset" has changed from "-07:00" to "-06:00".
The field "America/Shiprock.abbr" has changed from "MST" to "MDT".
The field "America/Shiprock.offset" has changed from "-07:00" to "-06:00".
The field "America/Yellowknife.abbr" has changed from "MST" to "MDT".
The field "America/Yellowknife.offset" has changed from "-07:00" to "-06:00".
The field "Canada/Mountain.abbr" has changed from "MST" to "MDT".
The field "Canada/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "MST7MDT.abbr" has changed from "MST" to "MDT".
The field "MST7MDT.offset" has changed from "-07:00" to "-06:00".
The field "Mexico/BajaSur.abbr" has changed from "MST" to "MDT".
The field "Mexico/BajaSur.offset" has changed from "-07:00" to "-06:00".
The field "Navajo.abbr" has changed from "MST" to "MDT".
The field "Navajo.offset" has changed from "-07:00" to "-06:00".
The field "US/Mountain.abbr" has changed from "MST" to "MDT".
The field "US/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "America/Bahia_Banderas.abbr" has changed from "CST" to "CDT".
The field "America/Bahia_Banderas.offset" has changed from "-06:00" to "-05:00".
The field "America/Chicago.abbr" has changed from "CST" to "CDT".
The field "America/Chicago.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Knox.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Knox.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Tell_City.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Tell_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Knox_IN.abbr" has changed from "CST" to "CDT".
The field "America/Knox_IN.offset" has changed from "-06:00" to "-05:00".
The field "America/Matamoros.abbr" has changed from "CST" to "CDT".
The field "America/Matamoros.offset" has changed from "-06:00" to "-05:00".
The field "America/Menominee.abbr" has changed from "CST" to "CDT".
The field "America/Menominee.offset" has changed from "-06:00" to "-05:00".
The field "America/Merida.abbr" has changed from "CST" to "CDT".
The field "America/Merida.offset" has changed from "-06:00" to "-05:00".
The field "America/Mexico_City.abbr" has changed from "CST" to "CDT".
The field "America/Mexico_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Monterrey.abbr" has changed from "CST" to "CDT".
The field "America/Monterrey.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Beulah.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Beulah.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Center.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Center.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/New_Salem.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/New_Salem.offset" has changed from "-06:00" to "-05:00".
The field "America/Rainy_River.abbr" has changed from "CST" to "CDT".
The field "America/Rainy_River.offset" has changed from "-06:00" to "-05:00".
The field "America/Rankin_Inlet.abbr" has changed from "CST" to "CDT".
The field "America/Rankin_Inlet.offset" has changed from "-06:00" to "-05:00".
The field "America/Resolute.abbr" has changed from "CST" to "CDT".
The field "America/Resolute.offset" has changed from "-06:00" to "-05:00".
The field "America/Winnipeg.abbr" has changed from "CST" to "CDT".
The field "America/Winnipeg.offset" has changed from "-06:00" to "-05:00".
The field "CST6CDT.abbr" has changed from "CST" to "CDT".
The field "CST6CDT.offset" has changed from "-06:00" to "-05:00".
The field "Canada/Central.abbr" has changed from "CST" to "CDT".
The field "Canada/Central.offset" has changed from "-06:00" to "-05:00".
The field "Mexico/General.abbr" has changed from "CST" to "CDT".
The field "Mexico/General.offset" has changed from "-06:00" to "-05:00".
The field "US/Central.abbr" has changed from "CST" to "CDT".
The field "US/Central.offset" has changed from "-06:00" to "-05:00".
The field "US/Indiana-Starke.abbr" has changed from "CST" to "CDT".
The field "US/Indiana-Starke.offset" has changed from "-06:00" to "-05:00".
The field "America/Detroit.abbr" has changed from "EST" to "EDT".
The field "America/Detroit.offset" has changed from "-05:00" to "-04:00".
The field "America/Fort_Wayne.abbr" has changed from "EST" to "EDT".
The field "America/Fort_Wayne.offset" has changed from "-05:00" to "-04:00".
The field "America/Grand_Turk.abbr" has changed from "EST" to "EDT".
The field "America/Grand_Turk.offset" has changed from "-05:00" to "-04:00".
The field "America/Havana.abbr" has changed from "CST" to "CDT".
The field "America/Havana.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Marengo.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Marengo.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Petersburg.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Petersburg.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vevay.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vevay.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vincennes.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vincennes.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Winamac.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Winamac.offset" has changed from "-05:00" to "-04:00".
The field "America/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Iqaluit.abbr" has changed from "EST" to "EDT".
The field "America/Iqaluit.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Monticello.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Monticello.offset" has changed from "-05:00" to "-04:00".
The field "America/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Montreal.abbr" has changed from "EST" to "EDT".
The field "America/Montreal.offset" has changed from "-05:00" to "-04:00".
The field "America/Nassau.abbr" has changed from "EST" to "EDT".
The field "America/Nassau.offset" has changed from "-05:00" to "-04:00".
The field "America/New_York.abbr" has changed from "EST" to "EDT".
The field "America/New_York.offset" has changed from "-05:00" to "-04:00".
The field "America/Nipigon.abbr" has changed from "EST" to "EDT".
The field "America/Nipigon.offset" has changed from "-05:00" to "-04:00".
The field "America/Pangnirtung.abbr" has changed from "EST" to "EDT".
The field "America/Pangnirtung.offset" has changed from "-05:00" to "-04:00".
The field "America/Port-au-Prince.abbr" has changed from "EST" to "EDT".
The field "America/Port-au-Prince.offset" has changed from "-05:00" to "-04:00".
The field "America/Thunder_Bay.abbr" has changed from "EST" to "EDT".
The field "America/Thunder_Bay.offset" has changed from "-05:00" to "-04:00".
The field "America/Toronto.abbr" has changed from "EST" to "EDT".
The field "America/Toronto.offset" has changed from "-05:00" to "-04:00".
The field "Canada/Eastern.abbr" has changed from "EST" to "EDT".
The field "Canada/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "Chile/EasterIsland.abbr" has changed from "-05" to "-06".
The field "Chile/EasterIsland.offset" has changed from "-05:00" to "-06:00".
The field "Cuba.abbr" has changed from "CST" to "CDT".
The field "Cuba.offset" has changed from "-05:00" to "-04:00".
The field "EST5EDT.abbr" has changed from "EST" to "EDT".
The field "EST5EDT.offset" has changed from "-05:00" to "-04:00".
The field "Pacific/Easter.abbr" has changed from "-05" to "-06".
The field "Pacific/Easter.offset" has changed from "-05:00" to "-06:00".
The field "US/East-Indiana.abbr" has changed from "EST" to "EDT".
The field "US/East-Indiana.offset" has changed from "-05:00" to "-04:00".
The field "US/Eastern.abbr" has changed from "EST" to "EDT".
The field "US/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "US/Michigan.abbr" has changed from "EST" to "EDT".
The field "US/Michigan.offset" has changed from "-05:00" to "-04:00".
The field "America/Glace_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Glace_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Goose_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Goose_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Halifax.abbr" has changed from "AST" to "ADT".
The field "America/Halifax.offset" has changed from "-04:00" to "-03:00".
The field "America/Moncton.abbr" has changed from "AST" to "ADT".
The field "America/Moncton.offset" has changed from "-04:00" to "-03:00".
The field "America/Thule.abbr" has changed from "AST" to "ADT".
The field "America/Thule.offset" has changed from "-04:00" to "-03:00".
The field "Atlantic/Bermuda.abbr" has changed from "AST" to "ADT".
The field "Atlantic/Bermuda.offset" has changed from "-04:00" to "-03:00".
The field "Canada/Atlantic.abbr" has changed from "AST" to "ADT".
The field "Canada/Atlantic.offset" has changed from "-04:00" to "-03:00".
The field "America/St_Johns.abbr" has changed from "NST" to "NDT".
The field "America/St_Johns.offset" has changed from "-03:30" to "-02:30".
The field "Canada/Newfoundland.abbr" has changed from "NST" to "NDT".
The field "Canada/Newfoundland.offset" has changed from "-03:30" to "-02:30".
The field "America/Asuncion.abbr" has changed from "-03" to "-04".
The field "America/Asuncion.offset" has changed from "-03:00" to "-04:00".
The field "America/Godthab.abbr" has changed from "-03" to "-02".
The field "America/Godthab.offset" has changed from "-03:00" to "-02:00".
The field "America/Miquelon.abbr" has changed from "-03" to "-02".
The field "America/Miquelon.offset" has changed from "-03:00" to "-02:00".
The field "America/Santiago.abbr" has changed from "-03" to "-04".
The field "America/Santiago.offset" has changed from "-03:00" to "-04:00".
The field "Chile/Continental.abbr" has changed from "-03" to "-04".
The field "Chile/Continental.offset" has changed from "-03:00" to "-04:00".
The field "America/Scoresbysund.abbr" has changed from "-01" to "+00".
The field "America/Scoresbysund.offset" has changed from "-01:00" to "+00:00".
The field "Atlantic/Azores.abbr" has changed from "-01" to "+00".
The field "Atlantic/Azores.offset" has changed from "-01:00" to "+00:00".
The field "Antarctica/Troll.abbr" has changed from "+00" to "+02".
The field "Antarctica/Troll.offset" has changed from "+00:00" to "+02:00".
The field "Atlantic/Canary.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Canary.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faeroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faeroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Madeira.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Madeira.offset" has changed from "+00:00" to "+01:00".
The field "Eire.abbr" has changed from "GMT" to "IST".
The field "Eire.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Belfast.abbr" has changed from "GMT" to "BST".
The field "Europe/Belfast.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Dublin.abbr" has changed from "GMT" to "IST".
The field "Europe/Dublin.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Guernsey.abbr" has changed from "GMT" to "BST".
The field "Europe/Guernsey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Isle_of_Man.abbr" has changed from "GMT" to "BST".
The field "Europe/Isle_of_Man.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Jersey.abbr" has changed from "GMT" to "BST".
The field "Europe/Jersey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Lisbon.abbr" has changed from "WET" to "WEST".
The field "Europe/Lisbon.offset" has changed from "+00:00" to "+01:00".
The field "Europe/London.abbr" has changed from "GMT" to "BST".
The field "Europe/London.offset" has changed from "+00:00" to "+01:00".
The field "GB.abbr" has changed from "GMT" to "BST".
The field "GB.offset" has changed from "+00:00" to "+01:00".
The field "GB-Eire.abbr" has changed from "GMT" to "BST".
The field "GB-Eire.offset" has changed from "+00:00" to "+01:00".
The field "Portugal.abbr" has changed from "WET" to "WEST".
The field "Portugal.offset" has changed from "+00:00" to "+01:00".
The field "WET.abbr" has changed from "WET" to "WEST".
The field "WET.offset" has changed from "+00:00" to "+01:00".
The field "Africa/Ceuta.abbr" has changed from "CET" to "CEST".
The field "Africa/Ceuta.offset" has changed from "+01:00" to "+02:00".
The field "Arctic/Longyearbyen.abbr" has changed from "CET" to "CEST".
The field "Arctic/Longyearbyen.offset" has changed from "+01:00" to "+02:00".
The field "Atlantic/Jan_Mayen.abbr" has changed from "CET" to "CEST".
The field "Atlantic/Jan_Mayen.offset" has changed from "+01:00" to "+02:00".
The field "CET.abbr" has changed from "CET" to "CEST".
The field "CET.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Amsterdam.abbr" has changed from "CET" to "CEST".
The field "Europe/Amsterdam.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Andorra.abbr" has changed from "CET" to "CEST".
The field "Europe/Andorra.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Belgrade.abbr" has changed from "CET" to "CEST".
The field "Europe/Belgrade.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Berlin.abbr" has changed from "CET" to "CEST".
The field "Europe/Berlin.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Bratislava.abbr" has changed from "CET" to "CEST".
The field "Europe/Bratislava.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Brussels.abbr" has changed from "CET" to "CEST".
The field "Europe/Brussels.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Budapest.abbr" has changed from "CET" to "CEST".
The field "Europe/Budapest.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Busingen.abbr" has changed from "CET" to "CEST".
The field "Europe/Busingen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Copenhagen.abbr" has changed from "CET" to "CEST".
The field "Europe/Copenhagen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Gibraltar.abbr" has changed from "CET" to "CEST".
The field "Europe/Gibraltar.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Ljubljana.abbr" has changed from "CET" to "CEST".
The field "Europe/Ljubljana.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Luxembourg.abbr" has changed from "CET" to "CEST".
The field "Europe/Luxembourg.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Madrid.abbr" has changed from "CET" to "CEST".
The field "Europe/Madrid.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Malta.abbr" has changed from "CET" to "CEST".
The field "Europe/Malta.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Monaco.abbr" has changed from "CET" to "CEST".
The field "Europe/Monaco.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Oslo.abbr" has changed from "CET" to "CEST".
The field "Europe/Oslo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Paris.abbr" has changed from "CET" to "CEST".
The field "Europe/Paris.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Podgorica.abbr" has changed from "CET" to "CEST".
The field "Europe/Podgorica.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Prague.abbr" has changed from "CET" to "CEST".
The field "Europe/Prague.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Rome.abbr" has changed from "CET" to "CEST".
The field "Europe/Rome.offset" has changed from "+01:00" to "+02:00".
The field "Europe/San_Marino.abbr" has changed from "CET" to "CEST".
The field "Europe/San_Marino.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Sarajevo.abbr" has changed from "CET" to "CEST".
The field "Europe/Sarajevo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Skopje.abbr" has changed from "CET" to "CEST".
The field "Europe/Skopje.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Stockholm.abbr" has changed from "CET" to "CEST".
The field "Europe/Stockholm.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Tirane.abbr" has changed from "CET" to "CEST".
The field "Europe/Tirane.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vaduz.abbr" has changed from "CET" to "CEST".
The field "Europe/Vaduz.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vatican.abbr" has changed from "CET" to "CEST".
The field "Europe/Vatican.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vienna.abbr" has changed from "CET" to "CEST".
The field "Europe/Vienna.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Warsaw.abbr" has changed from "CET" to "CEST".
The field "Europe/Warsaw.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zagreb.abbr" has changed from "CET" to "CEST".
The field "Europe/Zagreb.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zurich.abbr" has changed from "CET" to "CEST".
The field "Europe/Zurich.offset" has changed from "+01:00" to "+02:00".
The field "MET.abbr" has changed from "MET" to "MEST".
The field "MET.offset" has changed from "+01:00" to "+02:00".
The field "Poland.abbr" has changed from "CET" to "CEST".
The field "Poland.offset" has changed from "+01:00" to "+02:00".
The field "Asia/Amman.abbr" has changed from "EET" to "EEST".
The field "Asia/Amman.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Beirut.abbr" has changed from "EET" to "EEST".
The field "Asia/Beirut.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Damascus.abbr" has changed from "EET" to "EEST".
The field "Asia/Damascus.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Famagusta.abbr" has changed from "EET" to "EEST".
The field "Asia/Famagusta.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Gaza.abbr" has changed from "EET" to "EEST".
The field "Asia/Gaza.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Hebron.abbr" has changed from "EET" to "EEST".
The field "Asia/Hebron.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Jerusalem.abbr" has changed from "IST" to "IDT".
The field "Asia/Jerusalem.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Asia/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tel_Aviv.abbr" has changed from "IST" to "IDT".
The field "Asia/Tel_Aviv.offset" has changed from "+02:00" to "+03:00".
The field "EET.abbr" has changed from "EET" to "EEST".
The field "EET.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Athens.abbr" has changed from "EET" to "EEST".
The field "Europe/Athens.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Bucharest.abbr" has changed from "EET" to "EEST".
The field "Europe/Bucharest.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Chisinau.abbr" has changed from "EET" to "EEST".
The field "Europe/Chisinau.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Helsinki.abbr" has changed from "EET" to "EEST".
The field "Europe/Helsinki.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Kiev.abbr" has changed from "EET" to "EEST".
The field "Europe/Kiev.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Mariehamn.abbr" has changed from "EET" to "EEST".
The field "Europe/Mariehamn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Europe/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Riga.abbr" has changed from "EET" to "EEST".
The field "Europe/Riga.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Sofia.abbr" has changed from "EET" to "EEST".
The field "Europe/Sofia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tallinn.abbr" has changed from "EET" to "EEST".
The field "Europe/Tallinn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tiraspol.abbr" has changed from "EET" to "EEST".
The field "Europe/Tiraspol.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Uzhgorod.abbr" has changed from "EET" to "EEST".
The field "Europe/Uzhgorod.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Vilnius.abbr" has changed from "EET" to "EEST".
The field "Europe/Vilnius.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Zaporozhye.abbr" has changed from "EET" to "EEST".
The field "Europe/Zaporozhye.offset" has changed from "+02:00" to "+03:00".
The field "Israel.abbr" has changed from "IST" to "IDT".
The field "Israel.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tehran.abbr" has changed from "+0330" to "+0430".
The field "Asia/Tehran.offset" has changed from "+03:30" to "+04:30".
The field "Iran.abbr" has changed from "+0330" to "+0430".
The field "Iran.offset" has changed from "+03:30" to "+04:30".
The field "Australia/Adelaide.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Adelaide.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Broken_Hill.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Broken_Hill.offset" has changed from "+10:30" to "+09:30".
The field "Australia/South.abbr" has changed from "ACDT" to "ACST".
The field "Australia/South.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Yancowinna.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Yancowinna.offset" has changed from "+10:30" to "+09:30".
The field "Australia/ACT.abbr" has changed from "AEDT" to "AEST".
The field "Australia/ACT.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Canberra.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Canberra.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Currie.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Currie.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Hobart.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Hobart.offset" has changed from "+11:00" to "+10:00".
The field "Australia/LHI.abbr" has changed from "+11" to "+1030".
The field "Australia/LHI.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Lord_Howe.abbr" has changed from "+11" to "+1030".
The field "Australia/Lord_Howe.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Melbourne.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Melbourne.offset" has changed from "+11:00" to "+10:00".
The field "Australia/NSW.abbr" has changed from "AEDT" to "AEST".
The field "Australia/NSW.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Sydney.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Sydney.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Tasmania.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Tasmania.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Victoria.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Victoria.offset" has changed from "+11:00" to "+10:00".
The field "Pacific/Norfolk.abbr" has changed from "+12" to "+11".
The field "Pacific/Norfolk.offset" has changed from "+12:00" to "+11:00".
The field "Antarctica/McMurdo.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/McMurdo.offset" has changed from "+13:00" to "+12:00".
The field "Antarctica/South_Pole.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/South_Pole.offset" has changed from "+13:00" to "+12:00".
The field "NZ.abbr" has changed from "NZDT" to "NZST".
The field "NZ.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Auckland.abbr" has changed from "NZDT" to "NZST".
The field "Pacific/Auckland.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Fiji.abbr" has changed from "+13" to "+12".
The field "Pacific/Fiji.offset" has changed from "+13:00" to "+12:00".
The field "NZ-CHAT.abbr" has changed from "+1345" to "+1245".
The field "NZ-CHAT.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Chatham.abbr" has changed from "+1345" to "+1245".
The field "Pacific/Chatham.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Apia.abbr" has changed from "+14" to "+13".
The field "Pacific/Apia.offset" has changed from "+14:00" to "+13:00".
The field "America/Nuuk" has been added ('{"name":"America/Nuuk","abbr":"-02","offset":"-02:00"}').


[timezone] Data generated with warnings for locale "es".

The field "America/Adak.abbr" has changed from "HST" to "HDT".
The field "America/Adak.offset" has changed from "-10:00" to "-09:00".
The field "America/Atka.abbr" has changed from "HST" to "HDT".
The field "America/Atka.offset" has changed from "-10:00" to "-09:00".
The field "US/Aleutian.abbr" has changed from "HST" to "HDT".
The field "US/Aleutian.offset" has changed from "-10:00" to "-09:00".
The field "America/Anchorage.abbr" has changed from "AKST" to "AKDT".
The field "America/Anchorage.offset" has changed from "-09:00" to "-08:00".
The field "America/Juneau.abbr" has changed from "AKST" to "AKDT".
The field "America/Juneau.offset" has changed from "-09:00" to "-08:00".
The field "America/Metlakatla.abbr" has changed from "AKST" to "AKDT".
The field "America/Metlakatla.offset" has changed from "-09:00" to "-08:00".
The field "America/Nome.abbr" has changed from "AKST" to "AKDT".
The field "America/Nome.offset" has changed from "-09:00" to "-08:00".
The field "America/Sitka.abbr" has changed from "AKST" to "AKDT".
The field "America/Sitka.offset" has changed from "-09:00" to "-08:00".
The field "America/Yakutat.abbr" has changed from "AKST" to "AKDT".
The field "America/Yakutat.offset" has changed from "-09:00" to "-08:00".
The field "US/Alaska.abbr" has changed from "AKST" to "AKDT".
The field "US/Alaska.offset" has changed from "-09:00" to "-08:00".
The field "America/Dawson.abbr" has changed from "PST" to "MST".
The field "America/Dawson.offset" has changed from "-08:00" to "-07:00".
The field "America/Ensenada.abbr" has changed from "PST" to "PDT".
The field "America/Ensenada.offset" has changed from "-08:00" to "-07:00".
The field "America/Los_Angeles.abbr" has changed from "PST" to "PDT".
The field "America/Los_Angeles.offset" has changed from "-08:00" to "-07:00".
The field "America/Santa_Isabel.abbr" has changed from "PST" to "PDT".
The field "America/Santa_Isabel.offset" has changed from "-08:00" to "-07:00".
The field "America/Tijuana.abbr" has changed from "PST" to "PDT".
The field "America/Tijuana.offset" has changed from "-08:00" to "-07:00".
The field "America/Vancouver.abbr" has changed from "PST" to "PDT".
The field "America/Vancouver.offset" has changed from "-08:00" to "-07:00".
The field "America/Whitehorse.abbr" has changed from "PST" to "MST".
The field "America/Whitehorse.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Pacific.abbr" has changed from "PST" to "PDT".
The field "Canada/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Yukon.abbr" has changed from "PST" to "MST".
The field "Canada/Yukon.offset" has changed from "-08:00" to "-07:00".
The field "Mexico/BajaNorte.abbr" has changed from "PST" to "PDT".
The field "Mexico/BajaNorte.offset" has changed from "-08:00" to "-07:00".
The field "PST8PDT.abbr" has changed from "PST" to "PDT".
The field "PST8PDT.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific.abbr" has changed from "PST" to "PDT".
The field "US/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific-New.abbr" has changed from "PST" to "PDT".
The field "US/Pacific-New.offset" has changed from "-08:00" to "-07:00".
The field "America/Boise.abbr" has changed from "MST" to "MDT".
The field "America/Boise.offset" has changed from "-07:00" to "-06:00".
The field "America/Cambridge_Bay.abbr" has changed from "MST" to "MDT".
The field "America/Cambridge_Bay.offset" has changed from "-07:00" to "-06:00".
The field "America/Chihuahua.abbr" has changed from "MST" to "MDT".
The field "America/Chihuahua.offset" has changed from "-07:00" to "-06:00".
The field "America/Denver.abbr" has changed from "MST" to "MDT".
The field "America/Denver.offset" has changed from "-07:00" to "-06:00".
The field "America/Edmonton.abbr" has changed from "MST" to "MDT".
The field "America/Edmonton.offset" has changed from "-07:00" to "-06:00".
The field "America/Inuvik.abbr" has changed from "MST" to "MDT".
The field "America/Inuvik.offset" has changed from "-07:00" to "-06:00".
The field "America/Mazatlan.abbr" has changed from "MST" to "MDT".
The field "America/Mazatlan.offset" has changed from "-07:00" to "-06:00".
The field "America/Ojinaga.abbr" has changed from "MST" to "MDT".
The field "America/Ojinaga.offset" has changed from "-07:00" to "-06:00".
The field "America/Shiprock.abbr" has changed from "MST" to "MDT".
The field "America/Shiprock.offset" has changed from "-07:00" to "-06:00".
The field "America/Yellowknife.abbr" has changed from "MST" to "MDT".
The field "America/Yellowknife.offset" has changed from "-07:00" to "-06:00".
The field "Canada/Mountain.abbr" has changed from "MST" to "MDT".
The field "Canada/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "MST7MDT.abbr" has changed from "MST" to "MDT".
The field "MST7MDT.offset" has changed from "-07:00" to "-06:00".
The field "Mexico/BajaSur.abbr" has changed from "MST" to "MDT".
The field "Mexico/BajaSur.offset" has changed from "-07:00" to "-06:00".
The field "Navajo.abbr" has changed from "MST" to "MDT".
The field "Navajo.offset" has changed from "-07:00" to "-06:00".
The field "US/Mountain.abbr" has changed from "MST" to "MDT".
The field "US/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "America/Bahia_Banderas.abbr" has changed from "CST" to "CDT".
The field "America/Bahia_Banderas.offset" has changed from "-06:00" to "-05:00".
The field "America/Chicago.abbr" has changed from "CST" to "CDT".
The field "America/Chicago.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Knox.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Knox.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Tell_City.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Tell_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Knox_IN.abbr" has changed from "CST" to "CDT".
The field "America/Knox_IN.offset" has changed from "-06:00" to "-05:00".
The field "America/Matamoros.abbr" has changed from "CST" to "CDT".
The field "America/Matamoros.offset" has changed from "-06:00" to "-05:00".
The field "America/Menominee.abbr" has changed from "CST" to "CDT".
The field "America/Menominee.offset" has changed from "-06:00" to "-05:00".
The field "America/Merida.abbr" has changed from "CST" to "CDT".
The field "America/Merida.offset" has changed from "-06:00" to "-05:00".
The field "America/Mexico_City.abbr" has changed from "CST" to "CDT".
The field "America/Mexico_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Monterrey.abbr" has changed from "CST" to "CDT".
The field "America/Monterrey.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Beulah.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Beulah.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Center.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Center.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/New_Salem.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/New_Salem.offset" has changed from "-06:00" to "-05:00".
The field "America/Rainy_River.abbr" has changed from "CST" to "CDT".
The field "America/Rainy_River.offset" has changed from "-06:00" to "-05:00".
The field "America/Rankin_Inlet.abbr" has changed from "CST" to "CDT".
The field "America/Rankin_Inlet.offset" has changed from "-06:00" to "-05:00".
The field "America/Resolute.abbr" has changed from "CST" to "CDT".
The field "America/Resolute.offset" has changed from "-06:00" to "-05:00".
The field "America/Winnipeg.abbr" has changed from "CST" to "CDT".
The field "America/Winnipeg.offset" has changed from "-06:00" to "-05:00".
The field "CST6CDT.abbr" has changed from "CST" to "CDT".
The field "CST6CDT.offset" has changed from "-06:00" to "-05:00".
The field "Canada/Central.abbr" has changed from "CST" to "CDT".
The field "Canada/Central.offset" has changed from "-06:00" to "-05:00".
The field "Mexico/General.abbr" has changed from "CST" to "CDT".
The field "Mexico/General.offset" has changed from "-06:00" to "-05:00".
The field "US/Central.abbr" has changed from "CST" to "CDT".
The field "US/Central.offset" has changed from "-06:00" to "-05:00".
The field "US/Indiana-Starke.abbr" has changed from "CST" to "CDT".
The field "US/Indiana-Starke.offset" has changed from "-06:00" to "-05:00".
The field "America/Detroit.abbr" has changed from "EST" to "EDT".
The field "America/Detroit.offset" has changed from "-05:00" to "-04:00".
The field "America/Fort_Wayne.abbr" has changed from "EST" to "EDT".
The field "America/Fort_Wayne.offset" has changed from "-05:00" to "-04:00".
The field "America/Grand_Turk.abbr" has changed from "EST" to "EDT".
The field "America/Grand_Turk.offset" has changed from "-05:00" to "-04:00".
The field "America/Havana.abbr" has changed from "CST" to "CDT".
The field "America/Havana.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Marengo.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Marengo.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Petersburg.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Petersburg.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vevay.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vevay.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vincennes.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vincennes.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Winamac.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Winamac.offset" has changed from "-05:00" to "-04:00".
The field "America/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Iqaluit.abbr" has changed from "EST" to "EDT".
The field "America/Iqaluit.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Monticello.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Monticello.offset" has changed from "-05:00" to "-04:00".
The field "America/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Montreal.abbr" has changed from "EST" to "EDT".
The field "America/Montreal.offset" has changed from "-05:00" to "-04:00".
The field "America/Nassau.abbr" has changed from "EST" to "EDT".
The field "America/Nassau.offset" has changed from "-05:00" to "-04:00".
The field "America/New_York.abbr" has changed from "EST" to "EDT".
The field "America/New_York.offset" has changed from "-05:00" to "-04:00".
The field "America/Nipigon.abbr" has changed from "EST" to "EDT".
The field "America/Nipigon.offset" has changed from "-05:00" to "-04:00".
The field "America/Pangnirtung.abbr" has changed from "EST" to "EDT".
The field "America/Pangnirtung.offset" has changed from "-05:00" to "-04:00".
The field "America/Port-au-Prince.abbr" has changed from "EST" to "EDT".
The field "America/Port-au-Prince.offset" has changed from "-05:00" to "-04:00".
The field "America/Thunder_Bay.abbr" has changed from "EST" to "EDT".
The field "America/Thunder_Bay.offset" has changed from "-05:00" to "-04:00".
The field "America/Toronto.abbr" has changed from "EST" to "EDT".
The field "America/Toronto.offset" has changed from "-05:00" to "-04:00".
The field "Canada/Eastern.abbr" has changed from "EST" to "EDT".
The field "Canada/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "Chile/EasterIsland.abbr" has changed from "-05" to "-06".
The field "Chile/EasterIsland.offset" has changed from "-05:00" to "-06:00".
The field "Cuba.abbr" has changed from "CST" to "CDT".
The field "Cuba.offset" has changed from "-05:00" to "-04:00".
The field "EST5EDT.abbr" has changed from "EST" to "EDT".
The field "EST5EDT.offset" has changed from "-05:00" to "-04:00".
The field "Pacific/Easter.abbr" has changed from "-05" to "-06".
The field "Pacific/Easter.offset" has changed from "-05:00" to "-06:00".
The field "US/East-Indiana.abbr" has changed from "EST" to "EDT".
The field "US/East-Indiana.offset" has changed from "-05:00" to "-04:00".
The field "US/Eastern.abbr" has changed from "EST" to "EDT".
The field "US/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "US/Michigan.abbr" has changed from "EST" to "EDT".
The field "US/Michigan.offset" has changed from "-05:00" to "-04:00".
The field "America/Glace_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Glace_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Goose_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Goose_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Halifax.abbr" has changed from "AST" to "ADT".
The field "America/Halifax.offset" has changed from "-04:00" to "-03:00".
The field "America/Moncton.abbr" has changed from "AST" to "ADT".
The field "America/Moncton.offset" has changed from "-04:00" to "-03:00".
The field "America/Thule.abbr" has changed from "AST" to "ADT".
The field "America/Thule.offset" has changed from "-04:00" to "-03:00".
The field "Atlantic/Bermuda.abbr" has changed from "AST" to "ADT".
The field "Atlantic/Bermuda.offset" has changed from "-04:00" to "-03:00".
The field "Canada/Atlantic.abbr" has changed from "AST" to "ADT".
The field "Canada/Atlantic.offset" has changed from "-04:00" to "-03:00".
The field "America/St_Johns.abbr" has changed from "NST" to "NDT".
The field "America/St_Johns.offset" has changed from "-03:30" to "-02:30".
The field "Canada/Newfoundland.abbr" has changed from "NST" to "NDT".
The field "Canada/Newfoundland.offset" has changed from "-03:30" to "-02:30".
The field "America/Asuncion.abbr" has changed from "-03" to "-04".
The field "America/Asuncion.offset" has changed from "-03:00" to "-04:00".
The field "America/Godthab.abbr" has changed from "-03" to "-02".
The field "America/Godthab.offset" has changed from "-03:00" to "-02:00".
The field "America/Miquelon.abbr" has changed from "-03" to "-02".
The field "America/Miquelon.offset" has changed from "-03:00" to "-02:00".
The field "America/Santiago.abbr" has changed from "-03" to "-04".
The field "America/Santiago.offset" has changed from "-03:00" to "-04:00".
The field "Chile/Continental.abbr" has changed from "-03" to "-04".
The field "Chile/Continental.offset" has changed from "-03:00" to "-04:00".
The field "America/Scoresbysund.abbr" has changed from "-01" to "+00".
The field "America/Scoresbysund.offset" has changed from "-01:00" to "+00:00".
The field "Atlantic/Azores.abbr" has changed from "-01" to "+00".
The field "Atlantic/Azores.offset" has changed from "-01:00" to "+00:00".
The field "Antarctica/Troll.abbr" has changed from "+00" to "+02".
The field "Antarctica/Troll.offset" has changed from "+00:00" to "+02:00".
The field "Atlantic/Canary.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Canary.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faeroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faeroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Madeira.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Madeira.offset" has changed from "+00:00" to "+01:00".
The field "Eire.abbr" has changed from "GMT" to "IST".
The field "Eire.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Belfast.abbr" has changed from "GMT" to "BST".
The field "Europe/Belfast.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Dublin.abbr" has changed from "GMT" to "IST".
The field "Europe/Dublin.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Guernsey.abbr" has changed from "GMT" to "BST".
The field "Europe/Guernsey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Isle_of_Man.abbr" has changed from "GMT" to "BST".
The field "Europe/Isle_of_Man.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Jersey.abbr" has changed from "GMT" to "BST".
The field "Europe/Jersey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Lisbon.abbr" has changed from "WET" to "WEST".
The field "Europe/Lisbon.offset" has changed from "+00:00" to "+01:00".
The field "Europe/London.abbr" has changed from "GMT" to "BST".
The field "Europe/London.offset" has changed from "+00:00" to "+01:00".
The field "GB.abbr" has changed from "GMT" to "BST".
The field "GB.offset" has changed from "+00:00" to "+01:00".
The field "GB-Eire.abbr" has changed from "GMT" to "BST".
The field "GB-Eire.offset" has changed from "+00:00" to "+01:00".
The field "Portugal.abbr" has changed from "WET" to "WEST".
The field "Portugal.offset" has changed from "+00:00" to "+01:00".
The field "WET.abbr" has changed from "WET" to "WEST".
The field "WET.offset" has changed from "+00:00" to "+01:00".
The field "Africa/Ceuta.abbr" has changed from "CET" to "CEST".
The field "Africa/Ceuta.offset" has changed from "+01:00" to "+02:00".
The field "Arctic/Longyearbyen.abbr" has changed from "CET" to "CEST".
The field "Arctic/Longyearbyen.offset" has changed from "+01:00" to "+02:00".
The field "Atlantic/Jan_Mayen.abbr" has changed from "CET" to "CEST".
The field "Atlantic/Jan_Mayen.offset" has changed from "+01:00" to "+02:00".
The field "CET.abbr" has changed from "CET" to "CEST".
The field "CET.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Amsterdam.abbr" has changed from "CET" to "CEST".
The field "Europe/Amsterdam.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Andorra.abbr" has changed from "CET" to "CEST".
The field "Europe/Andorra.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Belgrade.abbr" has changed from "CET" to "CEST".
The field "Europe/Belgrade.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Berlin.abbr" has changed from "CET" to "CEST".
The field "Europe/Berlin.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Bratislava.abbr" has changed from "CET" to "CEST".
The field "Europe/Bratislava.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Brussels.abbr" has changed from "CET" to "CEST".
The field "Europe/Brussels.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Budapest.abbr" has changed from "CET" to "CEST".
The field "Europe/Budapest.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Busingen.abbr" has changed from "CET" to "CEST".
The field "Europe/Busingen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Copenhagen.abbr" has changed from "CET" to "CEST".
The field "Europe/Copenhagen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Gibraltar.abbr" has changed from "CET" to "CEST".
The field "Europe/Gibraltar.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Ljubljana.abbr" has changed from "CET" to "CEST".
The field "Europe/Ljubljana.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Luxembourg.abbr" has changed from "CET" to "CEST".
The field "Europe/Luxembourg.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Madrid.abbr" has changed from "CET" to "CEST".
The field "Europe/Madrid.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Malta.abbr" has changed from "CET" to "CEST".
The field "Europe/Malta.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Monaco.abbr" has changed from "CET" to "CEST".
The field "Europe/Monaco.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Oslo.abbr" has changed from "CET" to "CEST".
The field "Europe/Oslo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Paris.abbr" has changed from "CET" to "CEST".
The field "Europe/Paris.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Podgorica.abbr" has changed from "CET" to "CEST".
The field "Europe/Podgorica.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Prague.abbr" has changed from "CET" to "CEST".
The field "Europe/Prague.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Rome.abbr" has changed from "CET" to "CEST".
The field "Europe/Rome.offset" has changed from "+01:00" to "+02:00".
The field "Europe/San_Marino.abbr" has changed from "CET" to "CEST".
The field "Europe/San_Marino.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Sarajevo.abbr" has changed from "CET" to "CEST".
The field "Europe/Sarajevo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Skopje.abbr" has changed from "CET" to "CEST".
The field "Europe/Skopje.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Stockholm.abbr" has changed from "CET" to "CEST".
The field "Europe/Stockholm.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Tirane.abbr" has changed from "CET" to "CEST".
The field "Europe/Tirane.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vaduz.abbr" has changed from "CET" to "CEST".
The field "Europe/Vaduz.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vatican.abbr" has changed from "CET" to "CEST".
The field "Europe/Vatican.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vienna.abbr" has changed from "CET" to "CEST".
The field "Europe/Vienna.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Warsaw.abbr" has changed from "CET" to "CEST".
The field "Europe/Warsaw.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zagreb.abbr" has changed from "CET" to "CEST".
The field "Europe/Zagreb.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zurich.abbr" has changed from "CET" to "CEST".
The field "Europe/Zurich.offset" has changed from "+01:00" to "+02:00".
The field "MET.abbr" has changed from "MET" to "MEST".
The field "MET.offset" has changed from "+01:00" to "+02:00".
The field "Poland.abbr" has changed from "CET" to "CEST".
The field "Poland.offset" has changed from "+01:00" to "+02:00".
The field "Asia/Amman.abbr" has changed from "EET" to "EEST".
The field "Asia/Amman.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Beirut.abbr" has changed from "EET" to "EEST".
The field "Asia/Beirut.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Damascus.abbr" has changed from "EET" to "EEST".
The field "Asia/Damascus.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Famagusta.abbr" has changed from "EET" to "EEST".
The field "Asia/Famagusta.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Gaza.abbr" has changed from "EET" to "EEST".
The field "Asia/Gaza.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Hebron.abbr" has changed from "EET" to "EEST".
The field "Asia/Hebron.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Jerusalem.abbr" has changed from "IST" to "IDT".
The field "Asia/Jerusalem.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Asia/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tel_Aviv.abbr" has changed from "IST" to "IDT".
The field "Asia/Tel_Aviv.offset" has changed from "+02:00" to "+03:00".
The field "EET.abbr" has changed from "EET" to "EEST".
The field "EET.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Athens.abbr" has changed from "EET" to "EEST".
The field "Europe/Athens.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Bucharest.abbr" has changed from "EET" to "EEST".
The field "Europe/Bucharest.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Chisinau.abbr" has changed from "EET" to "EEST".
The field "Europe/Chisinau.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Helsinki.abbr" has changed from "EET" to "EEST".
The field "Europe/Helsinki.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Kiev.abbr" has changed from "EET" to "EEST".
The field "Europe/Kiev.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Mariehamn.abbr" has changed from "EET" to "EEST".
The field "Europe/Mariehamn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Europe/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Riga.abbr" has changed from "EET" to "EEST".
The field "Europe/Riga.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Sofia.abbr" has changed from "EET" to "EEST".
The field "Europe/Sofia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tallinn.abbr" has changed from "EET" to "EEST".
The field "Europe/Tallinn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tiraspol.abbr" has changed from "EET" to "EEST".
The field "Europe/Tiraspol.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Uzhgorod.abbr" has changed from "EET" to "EEST".
The field "Europe/Uzhgorod.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Vilnius.abbr" has changed from "EET" to "EEST".
The field "Europe/Vilnius.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Zaporozhye.abbr" has changed from "EET" to "EEST".
The field "Europe/Zaporozhye.offset" has changed from "+02:00" to "+03:00".
The field "Israel.abbr" has changed from "IST" to "IDT".
The field "Israel.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tehran.abbr" has changed from "+0330" to "+0430".
The field "Asia/Tehran.offset" has changed from "+03:30" to "+04:30".
The field "Iran.abbr" has changed from "+0330" to "+0430".
The field "Iran.offset" has changed from "+03:30" to "+04:30".
The field "Australia/Adelaide.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Adelaide.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Broken_Hill.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Broken_Hill.offset" has changed from "+10:30" to "+09:30".
The field "Australia/South.abbr" has changed from "ACDT" to "ACST".
The field "Australia/South.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Yancowinna.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Yancowinna.offset" has changed from "+10:30" to "+09:30".
The field "Australia/ACT.abbr" has changed from "AEDT" to "AEST".
The field "Australia/ACT.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Canberra.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Canberra.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Currie.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Currie.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Hobart.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Hobart.offset" has changed from "+11:00" to "+10:00".
The field "Australia/LHI.abbr" has changed from "+11" to "+1030".
The field "Australia/LHI.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Lord_Howe.abbr" has changed from "+11" to "+1030".
The field "Australia/Lord_Howe.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Melbourne.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Melbourne.offset" has changed from "+11:00" to "+10:00".
The field "Australia/NSW.abbr" has changed from "AEDT" to "AEST".
The field "Australia/NSW.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Sydney.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Sydney.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Tasmania.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Tasmania.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Victoria.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Victoria.offset" has changed from "+11:00" to "+10:00".
The field "Pacific/Norfolk.abbr" has changed from "+12" to "+11".
The field "Pacific/Norfolk.offset" has changed from "+12:00" to "+11:00".
The field "Antarctica/McMurdo.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/McMurdo.offset" has changed from "+13:00" to "+12:00".
The field "Antarctica/South_Pole.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/South_Pole.offset" has changed from "+13:00" to "+12:00".
The field "NZ.abbr" has changed from "NZDT" to "NZST".
The field "NZ.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Auckland.abbr" has changed from "NZDT" to "NZST".
The field "Pacific/Auckland.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Fiji.abbr" has changed from "+13" to "+12".
The field "Pacific/Fiji.offset" has changed from "+13:00" to "+12:00".
The field "NZ-CHAT.abbr" has changed from "+1345" to "+1245".
The field "NZ-CHAT.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Chatham.abbr" has changed from "+1345" to "+1245".
The field "Pacific/Chatham.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Apia.abbr" has changed from "+14" to "+13".
The field "Pacific/Apia.offset" has changed from "+14:00" to "+13:00".
The field "America/Nuuk" has been added ('{"name":"America/Nuuk","abbr":"-02","offset":"-02:00"}').


[timezone] Data generated with warnings for locale "fr-FR".

The field "America/Adak.abbr" has changed from "HST" to "HDT".
The field "America/Adak.offset" has changed from "-10:00" to "-09:00".
The field "America/Atka.abbr" has changed from "HST" to "HDT".
The field "America/Atka.offset" has changed from "-10:00" to "-09:00".
The field "US/Aleutian.abbr" has changed from "HST" to "HDT".
The field "US/Aleutian.offset" has changed from "-10:00" to "-09:00".
The field "America/Anchorage.abbr" has changed from "AKST" to "AKDT".
The field "America/Anchorage.offset" has changed from "-09:00" to "-08:00".
The field "America/Juneau.abbr" has changed from "AKST" to "AKDT".
The field "America/Juneau.offset" has changed from "-09:00" to "-08:00".
The field "America/Metlakatla.abbr" has changed from "AKST" to "AKDT".
The field "America/Metlakatla.offset" has changed from "-09:00" to "-08:00".
The field "America/Nome.abbr" has changed from "AKST" to "AKDT".
The field "America/Nome.offset" has changed from "-09:00" to "-08:00".
The field "America/Sitka.abbr" has changed from "AKST" to "AKDT".
The field "America/Sitka.offset" has changed from "-09:00" to "-08:00".
The field "America/Yakutat.abbr" has changed from "AKST" to "AKDT".
The field "America/Yakutat.offset" has changed from "-09:00" to "-08:00".
The field "US/Alaska.abbr" has changed from "AKST" to "AKDT".
The field "US/Alaska.offset" has changed from "-09:00" to "-08:00".
The field "America/Dawson.abbr" has changed from "PST" to "MST".
The field "America/Dawson.offset" has changed from "-08:00" to "-07:00".
The field "America/Ensenada.abbr" has changed from "PST" to "PDT".
The field "America/Ensenada.offset" has changed from "-08:00" to "-07:00".
The field "America/Los_Angeles.abbr" has changed from "PST" to "PDT".
The field "America/Los_Angeles.offset" has changed from "-08:00" to "-07:00".
The field "America/Santa_Isabel.abbr" has changed from "PST" to "PDT".
The field "America/Santa_Isabel.offset" has changed from "-08:00" to "-07:00".
The field "America/Tijuana.abbr" has changed from "PST" to "PDT".
The field "America/Tijuana.offset" has changed from "-08:00" to "-07:00".
The field "America/Vancouver.abbr" has changed from "PST" to "PDT".
The field "America/Vancouver.offset" has changed from "-08:00" to "-07:00".
The field "America/Whitehorse.abbr" has changed from "PST" to "MST".
The field "America/Whitehorse.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Pacific.abbr" has changed from "PST" to "PDT".
The field "Canada/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Yukon.abbr" has changed from "PST" to "MST".
The field "Canada/Yukon.offset" has changed from "-08:00" to "-07:00".
The field "Mexico/BajaNorte.abbr" has changed from "PST" to "PDT".
The field "Mexico/BajaNorte.offset" has changed from "-08:00" to "-07:00".
The field "PST8PDT.abbr" has changed from "PST" to "PDT".
The field "PST8PDT.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific.abbr" has changed from "PST" to "PDT".
The field "US/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific-New.abbr" has changed from "PST" to "PDT".
The field "US/Pacific-New.offset" has changed from "-08:00" to "-07:00".
The field "America/Boise.abbr" has changed from "MST" to "MDT".
The field "America/Boise.offset" has changed from "-07:00" to "-06:00".
The field "America/Cambridge_Bay.abbr" has changed from "MST" to "MDT".
The field "America/Cambridge_Bay.offset" has changed from "-07:00" to "-06:00".
The field "America/Chihuahua.abbr" has changed from "MST" to "MDT".
The field "America/Chihuahua.offset" has changed from "-07:00" to "-06:00".
The field "America/Denver.abbr" has changed from "MST" to "MDT".
The field "America/Denver.offset" has changed from "-07:00" to "-06:00".
The field "America/Edmonton.abbr" has changed from "MST" to "MDT".
The field "America/Edmonton.offset" has changed from "-07:00" to "-06:00".
The field "America/Inuvik.abbr" has changed from "MST" to "MDT".
The field "America/Inuvik.offset" has changed from "-07:00" to "-06:00".
The field "America/Mazatlan.abbr" has changed from "MST" to "MDT".
The field "America/Mazatlan.offset" has changed from "-07:00" to "-06:00".
The field "America/Ojinaga.abbr" has changed from "MST" to "MDT".
The field "America/Ojinaga.offset" has changed from "-07:00" to "-06:00".
The field "America/Shiprock.abbr" has changed from "MST" to "MDT".
The field "America/Shiprock.offset" has changed from "-07:00" to "-06:00".
The field "America/Yellowknife.abbr" has changed from "MST" to "MDT".
The field "America/Yellowknife.offset" has changed from "-07:00" to "-06:00".
The field "Canada/Mountain.abbr" has changed from "MST" to "MDT".
The field "Canada/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "MST7MDT.abbr" has changed from "MST" to "MDT".
The field "MST7MDT.offset" has changed from "-07:00" to "-06:00".
The field "Mexico/BajaSur.abbr" has changed from "MST" to "MDT".
The field "Mexico/BajaSur.offset" has changed from "-07:00" to "-06:00".
The field "Navajo.abbr" has changed from "MST" to "MDT".
The field "Navajo.offset" has changed from "-07:00" to "-06:00".
The field "US/Mountain.abbr" has changed from "MST" to "MDT".
The field "US/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "America/Bahia_Banderas.abbr" has changed from "CST" to "CDT".
The field "America/Bahia_Banderas.offset" has changed from "-06:00" to "-05:00".
The field "America/Chicago.abbr" has changed from "CST" to "CDT".
The field "America/Chicago.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Knox.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Knox.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Tell_City.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Tell_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Knox_IN.abbr" has changed from "CST" to "CDT".
The field "America/Knox_IN.offset" has changed from "-06:00" to "-05:00".
The field "America/Matamoros.abbr" has changed from "CST" to "CDT".
The field "America/Matamoros.offset" has changed from "-06:00" to "-05:00".
The field "America/Menominee.abbr" has changed from "CST" to "CDT".
The field "America/Menominee.offset" has changed from "-06:00" to "-05:00".
The field "America/Merida.abbr" has changed from "CST" to "CDT".
The field "America/Merida.offset" has changed from "-06:00" to "-05:00".
The field "America/Mexico_City.abbr" has changed from "CST" to "CDT".
The field "America/Mexico_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Monterrey.abbr" has changed from "CST" to "CDT".
The field "America/Monterrey.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Beulah.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Beulah.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Center.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Center.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/New_Salem.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/New_Salem.offset" has changed from "-06:00" to "-05:00".
The field "America/Rainy_River.abbr" has changed from "CST" to "CDT".
The field "America/Rainy_River.offset" has changed from "-06:00" to "-05:00".
The field "America/Rankin_Inlet.abbr" has changed from "CST" to "CDT".
The field "America/Rankin_Inlet.offset" has changed from "-06:00" to "-05:00".
The field "America/Resolute.abbr" has changed from "CST" to "CDT".
The field "America/Resolute.offset" has changed from "-06:00" to "-05:00".
The field "America/Winnipeg.abbr" has changed from "CST" to "CDT".
The field "America/Winnipeg.offset" has changed from "-06:00" to "-05:00".
The field "CST6CDT.abbr" has changed from "CST" to "CDT".
The field "CST6CDT.offset" has changed from "-06:00" to "-05:00".
The field "Canada/Central.abbr" has changed from "CST" to "CDT".
The field "Canada/Central.offset" has changed from "-06:00" to "-05:00".
The field "Mexico/General.abbr" has changed from "CST" to "CDT".
The field "Mexico/General.offset" has changed from "-06:00" to "-05:00".
The field "US/Central.abbr" has changed from "CST" to "CDT".
The field "US/Central.offset" has changed from "-06:00" to "-05:00".
The field "US/Indiana-Starke.abbr" has changed from "CST" to "CDT".
The field "US/Indiana-Starke.offset" has changed from "-06:00" to "-05:00".
The field "America/Detroit.abbr" has changed from "EST" to "EDT".
The field "America/Detroit.offset" has changed from "-05:00" to "-04:00".
The field "America/Fort_Wayne.abbr" has changed from "EST" to "EDT".
The field "America/Fort_Wayne.offset" has changed from "-05:00" to "-04:00".
The field "America/Grand_Turk.abbr" has changed from "EST" to "EDT".
The field "America/Grand_Turk.offset" has changed from "-05:00" to "-04:00".
The field "America/Havana.abbr" has changed from "CST" to "CDT".
The field "America/Havana.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Marengo.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Marengo.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Petersburg.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Petersburg.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vevay.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vevay.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vincennes.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vincennes.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Winamac.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Winamac.offset" has changed from "-05:00" to "-04:00".
The field "America/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Iqaluit.abbr" has changed from "EST" to "EDT".
The field "America/Iqaluit.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Monticello.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Monticello.offset" has changed from "-05:00" to "-04:00".
The field "America/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Montreal.abbr" has changed from "EST" to "EDT".
The field "America/Montreal.offset" has changed from "-05:00" to "-04:00".
The field "America/Nassau.abbr" has changed from "EST" to "EDT".
The field "America/Nassau.offset" has changed from "-05:00" to "-04:00".
The field "America/New_York.abbr" has changed from "EST" to "EDT".
The field "America/New_York.offset" has changed from "-05:00" to "-04:00".
The field "America/Nipigon.abbr" has changed from "EST" to "EDT".
The field "America/Nipigon.offset" has changed from "-05:00" to "-04:00".
The field "America/Pangnirtung.abbr" has changed from "EST" to "EDT".
The field "America/Pangnirtung.offset" has changed from "-05:00" to "-04:00".
The field "America/Port-au-Prince.abbr" has changed from "EST" to "EDT".
The field "America/Port-au-Prince.offset" has changed from "-05:00" to "-04:00".
The field "America/Thunder_Bay.abbr" has changed from "EST" to "EDT".
The field "America/Thunder_Bay.offset" has changed from "-05:00" to "-04:00".
The field "America/Toronto.abbr" has changed from "EST" to "EDT".
The field "America/Toronto.offset" has changed from "-05:00" to "-04:00".
The field "Canada/Eastern.abbr" has changed from "EST" to "EDT".
The field "Canada/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "Chile/EasterIsland.abbr" has changed from "-05" to "-06".
The field "Chile/EasterIsland.offset" has changed from "-05:00" to "-06:00".
The field "Cuba.abbr" has changed from "CST" to "CDT".
The field "Cuba.offset" has changed from "-05:00" to "-04:00".
The field "EST5EDT.abbr" has changed from "EST" to "EDT".
The field "EST5EDT.offset" has changed from "-05:00" to "-04:00".
The field "Pacific/Easter.abbr" has changed from "-05" to "-06".
The field "Pacific/Easter.offset" has changed from "-05:00" to "-06:00".
The field "US/East-Indiana.abbr" has changed from "EST" to "EDT".
The field "US/East-Indiana.offset" has changed from "-05:00" to "-04:00".
The field "US/Eastern.abbr" has changed from "EST" to "EDT".
The field "US/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "US/Michigan.abbr" has changed from "EST" to "EDT".
The field "US/Michigan.offset" has changed from "-05:00" to "-04:00".
The field "America/Glace_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Glace_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Goose_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Goose_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Halifax.abbr" has changed from "AST" to "ADT".
The field "America/Halifax.offset" has changed from "-04:00" to "-03:00".
The field "America/Moncton.abbr" has changed from "AST" to "ADT".
The field "America/Moncton.offset" has changed from "-04:00" to "-03:00".
The field "America/Thule.abbr" has changed from "AST" to "ADT".
The field "America/Thule.offset" has changed from "-04:00" to "-03:00".
The field "Atlantic/Bermuda.abbr" has changed from "AST" to "ADT".
The field "Atlantic/Bermuda.offset" has changed from "-04:00" to "-03:00".
The field "Canada/Atlantic.abbr" has changed from "AST" to "ADT".
The field "Canada/Atlantic.offset" has changed from "-04:00" to "-03:00".
The field "America/St_Johns.abbr" has changed from "NST" to "NDT".
The field "America/St_Johns.offset" has changed from "-03:30" to "-02:30".
The field "Canada/Newfoundland.abbr" has changed from "NST" to "NDT".
The field "Canada/Newfoundland.offset" has changed from "-03:30" to "-02:30".
The field "America/Asuncion.abbr" has changed from "-03" to "-04".
The field "America/Asuncion.offset" has changed from "-03:00" to "-04:00".
The field "America/Godthab.abbr" has changed from "-03" to "-02".
The field "America/Godthab.offset" has changed from "-03:00" to "-02:00".
The field "America/Miquelon.abbr" has changed from "-03" to "-02".
The field "America/Miquelon.offset" has changed from "-03:00" to "-02:00".
The field "America/Santiago.abbr" has changed from "-03" to "-04".
The field "America/Santiago.offset" has changed from "-03:00" to "-04:00".
The field "Chile/Continental.abbr" has changed from "-03" to "-04".
The field "Chile/Continental.offset" has changed from "-03:00" to "-04:00".
The field "America/Scoresbysund.abbr" has changed from "-01" to "+00".
The field "America/Scoresbysund.offset" has changed from "-01:00" to "+00:00".
The field "Atlantic/Azores.abbr" has changed from "-01" to "+00".
The field "Atlantic/Azores.offset" has changed from "-01:00" to "+00:00".
The field "Antarctica/Troll.abbr" has changed from "+00" to "+02".
The field "Antarctica/Troll.offset" has changed from "+00:00" to "+02:00".
The field "Atlantic/Canary.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Canary.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faeroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faeroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Madeira.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Madeira.offset" has changed from "+00:00" to "+01:00".
The field "Eire.abbr" has changed from "GMT" to "IST".
The field "Eire.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Belfast.abbr" has changed from "GMT" to "BST".
The field "Europe/Belfast.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Dublin.abbr" has changed from "GMT" to "IST".
The field "Europe/Dublin.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Guernsey.abbr" has changed from "GMT" to "BST".
The field "Europe/Guernsey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Isle_of_Man.abbr" has changed from "GMT" to "BST".
The field "Europe/Isle_of_Man.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Jersey.abbr" has changed from "GMT" to "BST".
The field "Europe/Jersey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Lisbon.abbr" has changed from "WET" to "WEST".
The field "Europe/Lisbon.offset" has changed from "+00:00" to "+01:00".
The field "Europe/London.abbr" has changed from "GMT" to "BST".
The field "Europe/London.offset" has changed from "+00:00" to "+01:00".
The field "GB.abbr" has changed from "GMT" to "BST".
The field "GB.offset" has changed from "+00:00" to "+01:00".
The field "GB-Eire.abbr" has changed from "GMT" to "BST".
The field "GB-Eire.offset" has changed from "+00:00" to "+01:00".
The field "Portugal.abbr" has changed from "WET" to "WEST".
The field "Portugal.offset" has changed from "+00:00" to "+01:00".
The field "WET.abbr" has changed from "WET" to "WEST".
The field "WET.offset" has changed from "+00:00" to "+01:00".
The field "Africa/Ceuta.abbr" has changed from "CET" to "CEST".
The field "Africa/Ceuta.offset" has changed from "+01:00" to "+02:00".
The field "Arctic/Longyearbyen.abbr" has changed from "CET" to "CEST".
The field "Arctic/Longyearbyen.offset" has changed from "+01:00" to "+02:00".
The field "Atlantic/Jan_Mayen.abbr" has changed from "CET" to "CEST".
The field "Atlantic/Jan_Mayen.offset" has changed from "+01:00" to "+02:00".
The field "CET.abbr" has changed from "CET" to "CEST".
The field "CET.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Amsterdam.abbr" has changed from "CET" to "CEST".
The field "Europe/Amsterdam.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Andorra.abbr" has changed from "CET" to "CEST".
The field "Europe/Andorra.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Belgrade.abbr" has changed from "CET" to "CEST".
The field "Europe/Belgrade.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Berlin.abbr" has changed from "CET" to "CEST".
The field "Europe/Berlin.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Bratislava.abbr" has changed from "CET" to "CEST".
The field "Europe/Bratislava.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Brussels.abbr" has changed from "CET" to "CEST".
The field "Europe/Brussels.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Budapest.abbr" has changed from "CET" to "CEST".
The field "Europe/Budapest.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Busingen.abbr" has changed from "CET" to "CEST".
The field "Europe/Busingen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Copenhagen.abbr" has changed from "CET" to "CEST".
The field "Europe/Copenhagen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Gibraltar.abbr" has changed from "CET" to "CEST".
The field "Europe/Gibraltar.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Ljubljana.abbr" has changed from "CET" to "CEST".
The field "Europe/Ljubljana.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Luxembourg.abbr" has changed from "CET" to "CEST".
The field "Europe/Luxembourg.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Madrid.abbr" has changed from "CET" to "CEST".
The field "Europe/Madrid.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Malta.abbr" has changed from "CET" to "CEST".
The field "Europe/Malta.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Monaco.abbr" has changed from "CET" to "CEST".
The field "Europe/Monaco.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Oslo.abbr" has changed from "CET" to "CEST".
The field "Europe/Oslo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Paris.abbr" has changed from "CET" to "CEST".
The field "Europe/Paris.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Podgorica.abbr" has changed from "CET" to "CEST".
The field "Europe/Podgorica.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Prague.abbr" has changed from "CET" to "CEST".
The field "Europe/Prague.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Rome.abbr" has changed from "CET" to "CEST".
The field "Europe/Rome.offset" has changed from "+01:00" to "+02:00".
The field "Europe/San_Marino.abbr" has changed from "CET" to "CEST".
The field "Europe/San_Marino.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Sarajevo.abbr" has changed from "CET" to "CEST".
The field "Europe/Sarajevo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Skopje.abbr" has changed from "CET" to "CEST".
The field "Europe/Skopje.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Stockholm.abbr" has changed from "CET" to "CEST".
The field "Europe/Stockholm.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Tirane.abbr" has changed from "CET" to "CEST".
The field "Europe/Tirane.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vaduz.abbr" has changed from "CET" to "CEST".
The field "Europe/Vaduz.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vatican.abbr" has changed from "CET" to "CEST".
The field "Europe/Vatican.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vienna.abbr" has changed from "CET" to "CEST".
The field "Europe/Vienna.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Warsaw.abbr" has changed from "CET" to "CEST".
The field "Europe/Warsaw.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zagreb.abbr" has changed from "CET" to "CEST".
The field "Europe/Zagreb.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zurich.abbr" has changed from "CET" to "CEST".
The field "Europe/Zurich.offset" has changed from "+01:00" to "+02:00".
The field "MET.abbr" has changed from "MET" to "MEST".
The field "MET.offset" has changed from "+01:00" to "+02:00".
The field "Poland.abbr" has changed from "CET" to "CEST".
The field "Poland.offset" has changed from "+01:00" to "+02:00".
The field "Asia/Amman.abbr" has changed from "EET" to "EEST".
The field "Asia/Amman.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Beirut.abbr" has changed from "EET" to "EEST".
The field "Asia/Beirut.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Damascus.abbr" has changed from "EET" to "EEST".
The field "Asia/Damascus.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Famagusta.abbr" has changed from "EET" to "EEST".
The field "Asia/Famagusta.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Gaza.abbr" has changed from "EET" to "EEST".
The field "Asia/Gaza.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Hebron.abbr" has changed from "EET" to "EEST".
The field "Asia/Hebron.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Jerusalem.abbr" has changed from "IST" to "IDT".
The field "Asia/Jerusalem.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Asia/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tel_Aviv.abbr" has changed from "IST" to "IDT".
The field "Asia/Tel_Aviv.offset" has changed from "+02:00" to "+03:00".
The field "EET.abbr" has changed from "EET" to "EEST".
The field "EET.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Athens.abbr" has changed from "EET" to "EEST".
The field "Europe/Athens.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Bucharest.abbr" has changed from "EET" to "EEST".
The field "Europe/Bucharest.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Chisinau.abbr" has changed from "EET" to "EEST".
The field "Europe/Chisinau.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Helsinki.abbr" has changed from "EET" to "EEST".
The field "Europe/Helsinki.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Kiev.abbr" has changed from "EET" to "EEST".
The field "Europe/Kiev.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Mariehamn.abbr" has changed from "EET" to "EEST".
The field "Europe/Mariehamn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Europe/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Riga.abbr" has changed from "EET" to "EEST".
The field "Europe/Riga.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Sofia.abbr" has changed from "EET" to "EEST".
The field "Europe/Sofia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tallinn.abbr" has changed from "EET" to "EEST".
The field "Europe/Tallinn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tiraspol.abbr" has changed from "EET" to "EEST".
The field "Europe/Tiraspol.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Uzhgorod.abbr" has changed from "EET" to "EEST".
The field "Europe/Uzhgorod.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Vilnius.abbr" has changed from "EET" to "EEST".
The field "Europe/Vilnius.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Zaporozhye.abbr" has changed from "EET" to "EEST".
The field "Europe/Zaporozhye.offset" has changed from "+02:00" to "+03:00".
The field "Israel.abbr" has changed from "IST" to "IDT".
The field "Israel.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tehran.abbr" has changed from "+0330" to "+0430".
The field "Asia/Tehran.offset" has changed from "+03:30" to "+04:30".
The field "Iran.abbr" has changed from "+0330" to "+0430".
The field "Iran.offset" has changed from "+03:30" to "+04:30".
The field "Australia/Adelaide.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Adelaide.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Broken_Hill.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Broken_Hill.offset" has changed from "+10:30" to "+09:30".
The field "Australia/South.abbr" has changed from "ACDT" to "ACST".
The field "Australia/South.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Yancowinna.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Yancowinna.offset" has changed from "+10:30" to "+09:30".
The field "Australia/ACT.abbr" has changed from "AEDT" to "AEST".
The field "Australia/ACT.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Canberra.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Canberra.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Currie.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Currie.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Hobart.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Hobart.offset" has changed from "+11:00" to "+10:00".
The field "Australia/LHI.abbr" has changed from "+11" to "+1030".
The field "Australia/LHI.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Lord_Howe.abbr" has changed from "+11" to "+1030".
The field "Australia/Lord_Howe.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Melbourne.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Melbourne.offset" has changed from "+11:00" to "+10:00".
The field "Australia/NSW.abbr" has changed from "AEDT" to "AEST".
The field "Australia/NSW.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Sydney.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Sydney.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Tasmania.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Tasmania.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Victoria.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Victoria.offset" has changed from "+11:00" to "+10:00".
The field "Pacific/Norfolk.abbr" has changed from "+12" to "+11".
The field "Pacific/Norfolk.offset" has changed from "+12:00" to "+11:00".
The field "Antarctica/McMurdo.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/McMurdo.offset" has changed from "+13:00" to "+12:00".
The field "Antarctica/South_Pole.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/South_Pole.offset" has changed from "+13:00" to "+12:00".
The field "NZ.abbr" has changed from "NZDT" to "NZST".
The field "NZ.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Auckland.abbr" has changed from "NZDT" to "NZST".
The field "Pacific/Auckland.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Fiji.abbr" has changed from "+13" to "+12".
The field "Pacific/Fiji.offset" has changed from "+13:00" to "+12:00".
The field "NZ-CHAT.abbr" has changed from "+1345" to "+1245".
The field "NZ-CHAT.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Chatham.abbr" has changed from "+1345" to "+1245".
The field "Pacific/Chatham.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Apia.abbr" has changed from "+14" to "+13".
The field "Pacific/Apia.offset" has changed from "+14:00" to "+13:00".
The field "America/Nuuk" has been added ('{"name":"America/Nuuk","abbr":"-02","offset":"-02:00"}').


[timezone] Data generated with warnings for locale "zh-CN".

The field "America/Adak.abbr" has changed from "HST" to "HDT".
The field "America/Adak.offset" has changed from "-10:00" to "-09:00".
The field "America/Atka.abbr" has changed from "HST" to "HDT".
The field "America/Atka.offset" has changed from "-10:00" to "-09:00".
The field "US/Aleutian.abbr" has changed from "HST" to "HDT".
The field "US/Aleutian.offset" has changed from "-10:00" to "-09:00".
The field "America/Anchorage.abbr" has changed from "AKST" to "AKDT".
The field "America/Anchorage.offset" has changed from "-09:00" to "-08:00".
The field "America/Juneau.abbr" has changed from "AKST" to "AKDT".
The field "America/Juneau.offset" has changed from "-09:00" to "-08:00".
The field "America/Metlakatla.abbr" has changed from "AKST" to "AKDT".
The field "America/Metlakatla.offset" has changed from "-09:00" to "-08:00".
The field "America/Nome.abbr" has changed from "AKST" to "AKDT".
The field "America/Nome.offset" has changed from "-09:00" to "-08:00".
The field "America/Sitka.abbr" has changed from "AKST" to "AKDT".
The field "America/Sitka.offset" has changed from "-09:00" to "-08:00".
The field "America/Yakutat.abbr" has changed from "AKST" to "AKDT".
The field "America/Yakutat.offset" has changed from "-09:00" to "-08:00".
The field "US/Alaska.abbr" has changed from "AKST" to "AKDT".
The field "US/Alaska.offset" has changed from "-09:00" to "-08:00".
The field "America/Dawson.abbr" has changed from "PST" to "MST".
The field "America/Dawson.offset" has changed from "-08:00" to "-07:00".
The field "America/Ensenada.abbr" has changed from "PST" to "PDT".
The field "America/Ensenada.offset" has changed from "-08:00" to "-07:00".
The field "America/Los_Angeles.abbr" has changed from "PST" to "PDT".
The field "America/Los_Angeles.offset" has changed from "-08:00" to "-07:00".
The field "America/Santa_Isabel.abbr" has changed from "PST" to "PDT".
The field "America/Santa_Isabel.offset" has changed from "-08:00" to "-07:00".
The field "America/Tijuana.abbr" has changed from "PST" to "PDT".
The field "America/Tijuana.offset" has changed from "-08:00" to "-07:00".
The field "America/Vancouver.abbr" has changed from "PST" to "PDT".
The field "America/Vancouver.offset" has changed from "-08:00" to "-07:00".
The field "America/Whitehorse.abbr" has changed from "PST" to "MST".
The field "America/Whitehorse.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Pacific.abbr" has changed from "PST" to "PDT".
The field "Canada/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Yukon.abbr" has changed from "PST" to "MST".
The field "Canada/Yukon.offset" has changed from "-08:00" to "-07:00".
The field "Mexico/BajaNorte.abbr" has changed from "PST" to "PDT".
The field "Mexico/BajaNorte.offset" has changed from "-08:00" to "-07:00".
The field "PST8PDT.abbr" has changed from "PST" to "PDT".
The field "PST8PDT.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific.abbr" has changed from "PST" to "PDT".
The field "US/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific-New.abbr" has changed from "PST" to "PDT".
The field "US/Pacific-New.offset" has changed from "-08:00" to "-07:00".
The field "America/Boise.abbr" has changed from "MST" to "MDT".
The field "America/Boise.offset" has changed from "-07:00" to "-06:00".
The field "America/Cambridge_Bay.abbr" has changed from "MST" to "MDT".
The field "America/Cambridge_Bay.offset" has changed from "-07:00" to "-06:00".
The field "America/Chihuahua.abbr" has changed from "MST" to "MDT".
The field "America/Chihuahua.offset" has changed from "-07:00" to "-06:00".
The field "America/Denver.abbr" has changed from "MST" to "MDT".
The field "America/Denver.offset" has changed from "-07:00" to "-06:00".
The field "America/Edmonton.abbr" has changed from "MST" to "MDT".
The field "America/Edmonton.offset" has changed from "-07:00" to "-06:00".
The field "America/Inuvik.abbr" has changed from "MST" to "MDT".
The field "America/Inuvik.offset" has changed from "-07:00" to "-06:00".
The field "America/Mazatlan.abbr" has changed from "MST" to "MDT".
The field "America/Mazatlan.offset" has changed from "-07:00" to "-06:00".
The field "America/Ojinaga.abbr" has changed from "MST" to "MDT".
The field "America/Ojinaga.offset" has changed from "-07:00" to "-06:00".
The field "America/Shiprock.abbr" has changed from "MST" to "MDT".
The field "America/Shiprock.offset" has changed from "-07:00" to "-06:00".
The field "America/Yellowknife.abbr" has changed from "MST" to "MDT".
The field "America/Yellowknife.offset" has changed from "-07:00" to "-06:00".
The field "Canada/Mountain.abbr" has changed from "MST" to "MDT".
The field "Canada/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "MST7MDT.abbr" has changed from "MST" to "MDT".
The field "MST7MDT.offset" has changed from "-07:00" to "-06:00".
The field "Mexico/BajaSur.abbr" has changed from "MST" to "MDT".
The field "Mexico/BajaSur.offset" has changed from "-07:00" to "-06:00".
The field "Navajo.abbr" has changed from "MST" to "MDT".
The field "Navajo.offset" has changed from "-07:00" to "-06:00".
The field "US/Mountain.abbr" has changed from "MST" to "MDT".
The field "US/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "America/Bahia_Banderas.abbr" has changed from "CST" to "CDT".
The field "America/Bahia_Banderas.offset" has changed from "-06:00" to "-05:00".
The field "America/Chicago.abbr" has changed from "CST" to "CDT".
The field "America/Chicago.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Knox.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Knox.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Tell_City.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Tell_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Knox_IN.abbr" has changed from "CST" to "CDT".
The field "America/Knox_IN.offset" has changed from "-06:00" to "-05:00".
The field "America/Matamoros.abbr" has changed from "CST" to "CDT".
The field "America/Matamoros.offset" has changed from "-06:00" to "-05:00".
The field "America/Menominee.abbr" has changed from "CST" to "CDT".
The field "America/Menominee.offset" has changed from "-06:00" to "-05:00".
The field "America/Merida.abbr" has changed from "CST" to "CDT".
The field "America/Merida.offset" has changed from "-06:00" to "-05:00".
The field "America/Mexico_City.abbr" has changed from "CST" to "CDT".
The field "America/Mexico_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Monterrey.abbr" has changed from "CST" to "CDT".
The field "America/Monterrey.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Beulah.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Beulah.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Center.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Center.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/New_Salem.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/New_Salem.offset" has changed from "-06:00" to "-05:00".
The field "America/Rainy_River.abbr" has changed from "CST" to "CDT".
The field "America/Rainy_River.offset" has changed from "-06:00" to "-05:00".
The field "America/Rankin_Inlet.abbr" has changed from "CST" to "CDT".
The field "America/Rankin_Inlet.offset" has changed from "-06:00" to "-05:00".
The field "America/Resolute.abbr" has changed from "CST" to "CDT".
The field "America/Resolute.offset" has changed from "-06:00" to "-05:00".
The field "America/Winnipeg.abbr" has changed from "CST" to "CDT".
The field "America/Winnipeg.offset" has changed from "-06:00" to "-05:00".
The field "CST6CDT.abbr" has changed from "CST" to "CDT".
The field "CST6CDT.offset" has changed from "-06:00" to "-05:00".
The field "Canada/Central.abbr" has changed from "CST" to "CDT".
The field "Canada/Central.offset" has changed from "-06:00" to "-05:00".
The field "Mexico/General.abbr" has changed from "CST" to "CDT".
The field "Mexico/General.offset" has changed from "-06:00" to "-05:00".
The field "US/Central.abbr" has changed from "CST" to "CDT".
The field "US/Central.offset" has changed from "-06:00" to "-05:00".
The field "US/Indiana-Starke.abbr" has changed from "CST" to "CDT".
The field "US/Indiana-Starke.offset" has changed from "-06:00" to "-05:00".
The field "America/Detroit.abbr" has changed from "EST" to "EDT".
The field "America/Detroit.offset" has changed from "-05:00" to "-04:00".
The field "America/Fort_Wayne.abbr" has changed from "EST" to "EDT".
The field "America/Fort_Wayne.offset" has changed from "-05:00" to "-04:00".
The field "America/Grand_Turk.abbr" has changed from "EST" to "EDT".
The field "America/Grand_Turk.offset" has changed from "-05:00" to "-04:00".
The field "America/Havana.abbr" has changed from "CST" to "CDT".
The field "America/Havana.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Marengo.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Marengo.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Petersburg.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Petersburg.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vevay.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vevay.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vincennes.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vincennes.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Winamac.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Winamac.offset" has changed from "-05:00" to "-04:00".
The field "America/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Iqaluit.abbr" has changed from "EST" to "EDT".
The field "America/Iqaluit.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Monticello.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Monticello.offset" has changed from "-05:00" to "-04:00".
The field "America/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Montreal.abbr" has changed from "EST" to "EDT".
The field "America/Montreal.offset" has changed from "-05:00" to "-04:00".
The field "America/Nassau.abbr" has changed from "EST" to "EDT".
The field "America/Nassau.offset" has changed from "-05:00" to "-04:00".
The field "America/New_York.abbr" has changed from "EST" to "EDT".
The field "America/New_York.offset" has changed from "-05:00" to "-04:00".
The field "America/Nipigon.abbr" has changed from "EST" to "EDT".
The field "America/Nipigon.offset" has changed from "-05:00" to "-04:00".
The field "America/Pangnirtung.abbr" has changed from "EST" to "EDT".
The field "America/Pangnirtung.offset" has changed from "-05:00" to "-04:00".
The field "America/Port-au-Prince.abbr" has changed from "EST" to "EDT".
The field "America/Port-au-Prince.offset" has changed from "-05:00" to "-04:00".
The field "America/Thunder_Bay.abbr" has changed from "EST" to "EDT".
The field "America/Thunder_Bay.offset" has changed from "-05:00" to "-04:00".
The field "America/Toronto.abbr" has changed from "EST" to "EDT".
The field "America/Toronto.offset" has changed from "-05:00" to "-04:00".
The field "Canada/Eastern.abbr" has changed from "EST" to "EDT".
The field "Canada/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "Chile/EasterIsland.abbr" has changed from "-05" to "-06".
The field "Chile/EasterIsland.offset" has changed from "-05:00" to "-06:00".
The field "Cuba.abbr" has changed from "CST" to "CDT".
The field "Cuba.offset" has changed from "-05:00" to "-04:00".
The field "EST5EDT.abbr" has changed from "EST" to "EDT".
The field "EST5EDT.offset" has changed from "-05:00" to "-04:00".
The field "Pacific/Easter.abbr" has changed from "-05" to "-06".
The field "Pacific/Easter.offset" has changed from "-05:00" to "-06:00".
The field "US/East-Indiana.abbr" has changed from "EST" to "EDT".
The field "US/East-Indiana.offset" has changed from "-05:00" to "-04:00".
The field "US/Eastern.abbr" has changed from "EST" to "EDT".
The field "US/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "US/Michigan.abbr" has changed from "EST" to "EDT".
The field "US/Michigan.offset" has changed from "-05:00" to "-04:00".
The field "America/Glace_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Glace_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Goose_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Goose_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Halifax.abbr" has changed from "AST" to "ADT".
The field "America/Halifax.offset" has changed from "-04:00" to "-03:00".
The field "America/Moncton.abbr" has changed from "AST" to "ADT".
The field "America/Moncton.offset" has changed from "-04:00" to "-03:00".
The field "America/Thule.abbr" has changed from "AST" to "ADT".
The field "America/Thule.offset" has changed from "-04:00" to "-03:00".
The field "Atlantic/Bermuda.abbr" has changed from "AST" to "ADT".
The field "Atlantic/Bermuda.offset" has changed from "-04:00" to "-03:00".
The field "Canada/Atlantic.abbr" has changed from "AST" to "ADT".
The field "Canada/Atlantic.offset" has changed from "-04:00" to "-03:00".
The field "America/St_Johns.abbr" has changed from "NST" to "NDT".
The field "America/St_Johns.offset" has changed from "-03:30" to "-02:30".
The field "Canada/Newfoundland.abbr" has changed from "NST" to "NDT".
The field "Canada/Newfoundland.offset" has changed from "-03:30" to "-02:30".
The field "America/Asuncion.abbr" has changed from "-03" to "-04".
The field "America/Asuncion.offset" has changed from "-03:00" to "-04:00".
The field "America/Godthab.abbr" has changed from "-03" to "-02".
The field "America/Godthab.offset" has changed from "-03:00" to "-02:00".
The field "America/Miquelon.abbr" has changed from "-03" to "-02".
The field "America/Miquelon.offset" has changed from "-03:00" to "-02:00".
The field "America/Santiago.abbr" has changed from "-03" to "-04".
The field "America/Santiago.offset" has changed from "-03:00" to "-04:00".
The field "Chile/Continental.abbr" has changed from "-03" to "-04".
The field "Chile/Continental.offset" has changed from "-03:00" to "-04:00".
The field "America/Scoresbysund.abbr" has changed from "-01" to "+00".
The field "America/Scoresbysund.offset" has changed from "-01:00" to "+00:00".
The field "Atlantic/Azores.abbr" has changed from "-01" to "+00".
The field "Atlantic/Azores.offset" has changed from "-01:00" to "+00:00".
The field "Antarctica/Troll.abbr" has changed from "+00" to "+02".
The field "Antarctica/Troll.offset" has changed from "+00:00" to "+02:00".
The field "Atlantic/Canary.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Canary.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faeroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faeroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Madeira.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Madeira.offset" has changed from "+00:00" to "+01:00".
The field "Eire.abbr" has changed from "GMT" to "IST".
The field "Eire.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Belfast.abbr" has changed from "GMT" to "BST".
The field "Europe/Belfast.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Dublin.abbr" has changed from "GMT" to "IST".
The field "Europe/Dublin.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Guernsey.abbr" has changed from "GMT" to "BST".
The field "Europe/Guernsey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Isle_of_Man.abbr" has changed from "GMT" to "BST".
The field "Europe/Isle_of_Man.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Jersey.abbr" has changed from "GMT" to "BST".
The field "Europe/Jersey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Lisbon.abbr" has changed from "WET" to "WEST".
The field "Europe/Lisbon.offset" has changed from "+00:00" to "+01:00".
The field "Europe/London.abbr" has changed from "GMT" to "BST".
The field "Europe/London.offset" has changed from "+00:00" to "+01:00".
The field "GB.abbr" has changed from "GMT" to "BST".
The field "GB.offset" has changed from "+00:00" to "+01:00".
The field "GB-Eire.abbr" has changed from "GMT" to "BST".
The field "GB-Eire.offset" has changed from "+00:00" to "+01:00".
The field "Portugal.abbr" has changed from "WET" to "WEST".
The field "Portugal.offset" has changed from "+00:00" to "+01:00".
The field "WET.abbr" has changed from "WET" to "WEST".
The field "WET.offset" has changed from "+00:00" to "+01:00".
The field "Africa/Ceuta.abbr" has changed from "CET" to "CEST".
The field "Africa/Ceuta.offset" has changed from "+01:00" to "+02:00".
The field "Arctic/Longyearbyen.abbr" has changed from "CET" to "CEST".
The field "Arctic/Longyearbyen.offset" has changed from "+01:00" to "+02:00".
The field "Atlantic/Jan_Mayen.abbr" has changed from "CET" to "CEST".
The field "Atlantic/Jan_Mayen.offset" has changed from "+01:00" to "+02:00".
The field "CET.abbr" has changed from "CET" to "CEST".
The field "CET.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Amsterdam.abbr" has changed from "CET" to "CEST".
The field "Europe/Amsterdam.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Andorra.abbr" has changed from "CET" to "CEST".
The field "Europe/Andorra.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Belgrade.abbr" has changed from "CET" to "CEST".
The field "Europe/Belgrade.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Berlin.abbr" has changed from "CET" to "CEST".
The field "Europe/Berlin.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Bratislava.abbr" has changed from "CET" to "CEST".
The field "Europe/Bratislava.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Brussels.abbr" has changed from "CET" to "CEST".
The field "Europe/Brussels.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Budapest.abbr" has changed from "CET" to "CEST".
The field "Europe/Budapest.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Busingen.abbr" has changed from "CET" to "CEST".
The field "Europe/Busingen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Copenhagen.abbr" has changed from "CET" to "CEST".
The field "Europe/Copenhagen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Gibraltar.abbr" has changed from "CET" to "CEST".
The field "Europe/Gibraltar.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Ljubljana.abbr" has changed from "CET" to "CEST".
The field "Europe/Ljubljana.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Luxembourg.abbr" has changed from "CET" to "CEST".
The field "Europe/Luxembourg.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Madrid.abbr" has changed from "CET" to "CEST".
The field "Europe/Madrid.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Malta.abbr" has changed from "CET" to "CEST".
The field "Europe/Malta.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Monaco.abbr" has changed from "CET" to "CEST".
The field "Europe/Monaco.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Oslo.abbr" has changed from "CET" to "CEST".
The field "Europe/Oslo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Paris.abbr" has changed from "CET" to "CEST".
The field "Europe/Paris.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Podgorica.abbr" has changed from "CET" to "CEST".
The field "Europe/Podgorica.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Prague.abbr" has changed from "CET" to "CEST".
The field "Europe/Prague.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Rome.abbr" has changed from "CET" to "CEST".
The field "Europe/Rome.offset" has changed from "+01:00" to "+02:00".
The field "Europe/San_Marino.abbr" has changed from "CET" to "CEST".
The field "Europe/San_Marino.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Sarajevo.abbr" has changed from "CET" to "CEST".
The field "Europe/Sarajevo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Skopje.abbr" has changed from "CET" to "CEST".
The field "Europe/Skopje.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Stockholm.abbr" has changed from "CET" to "CEST".
The field "Europe/Stockholm.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Tirane.abbr" has changed from "CET" to "CEST".
The field "Europe/Tirane.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vaduz.abbr" has changed from "CET" to "CEST".
The field "Europe/Vaduz.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vatican.abbr" has changed from "CET" to "CEST".
The field "Europe/Vatican.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vienna.abbr" has changed from "CET" to "CEST".
The field "Europe/Vienna.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Warsaw.abbr" has changed from "CET" to "CEST".
The field "Europe/Warsaw.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zagreb.abbr" has changed from "CET" to "CEST".
The field "Europe/Zagreb.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zurich.abbr" has changed from "CET" to "CEST".
The field "Europe/Zurich.offset" has changed from "+01:00" to "+02:00".
The field "MET.abbr" has changed from "MET" to "MEST".
The field "MET.offset" has changed from "+01:00" to "+02:00".
The field "Poland.abbr" has changed from "CET" to "CEST".
The field "Poland.offset" has changed from "+01:00" to "+02:00".
The field "Asia/Amman.abbr" has changed from "EET" to "EEST".
The field "Asia/Amman.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Beirut.abbr" has changed from "EET" to "EEST".
The field "Asia/Beirut.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Damascus.abbr" has changed from "EET" to "EEST".
The field "Asia/Damascus.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Famagusta.abbr" has changed from "EET" to "EEST".
The field "Asia/Famagusta.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Gaza.abbr" has changed from "EET" to "EEST".
The field "Asia/Gaza.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Hebron.abbr" has changed from "EET" to "EEST".
The field "Asia/Hebron.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Jerusalem.abbr" has changed from "IST" to "IDT".
The field "Asia/Jerusalem.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Asia/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tel_Aviv.abbr" has changed from "IST" to "IDT".
The field "Asia/Tel_Aviv.offset" has changed from "+02:00" to "+03:00".
The field "EET.abbr" has changed from "EET" to "EEST".
The field "EET.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Athens.abbr" has changed from "EET" to "EEST".
The field "Europe/Athens.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Bucharest.abbr" has changed from "EET" to "EEST".
The field "Europe/Bucharest.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Chisinau.abbr" has changed from "EET" to "EEST".
The field "Europe/Chisinau.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Helsinki.abbr" has changed from "EET" to "EEST".
The field "Europe/Helsinki.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Kiev.abbr" has changed from "EET" to "EEST".
The field "Europe/Kiev.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Mariehamn.abbr" has changed from "EET" to "EEST".
The field "Europe/Mariehamn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Europe/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Riga.abbr" has changed from "EET" to "EEST".
The field "Europe/Riga.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Sofia.abbr" has changed from "EET" to "EEST".
The field "Europe/Sofia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tallinn.abbr" has changed from "EET" to "EEST".
The field "Europe/Tallinn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tiraspol.abbr" has changed from "EET" to "EEST".
The field "Europe/Tiraspol.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Uzhgorod.abbr" has changed from "EET" to "EEST".
The field "Europe/Uzhgorod.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Vilnius.abbr" has changed from "EET" to "EEST".
The field "Europe/Vilnius.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Zaporozhye.abbr" has changed from "EET" to "EEST".
The field "Europe/Zaporozhye.offset" has changed from "+02:00" to "+03:00".
The field "Israel.abbr" has changed from "IST" to "IDT".
The field "Israel.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tehran.abbr" has changed from "+0330" to "+0430".
The field "Asia/Tehran.offset" has changed from "+03:30" to "+04:30".
The field "Iran.abbr" has changed from "+0330" to "+0430".
The field "Iran.offset" has changed from "+03:30" to "+04:30".
The field "Australia/Adelaide.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Adelaide.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Broken_Hill.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Broken_Hill.offset" has changed from "+10:30" to "+09:30".
The field "Australia/South.abbr" has changed from "ACDT" to "ACST".
The field "Australia/South.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Yancowinna.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Yancowinna.offset" has changed from "+10:30" to "+09:30".
The field "Australia/ACT.abbr" has changed from "AEDT" to "AEST".
The field "Australia/ACT.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Canberra.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Canberra.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Currie.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Currie.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Hobart.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Hobart.offset" has changed from "+11:00" to "+10:00".
The field "Australia/LHI.abbr" has changed from "+11" to "+1030".
The field "Australia/LHI.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Lord_Howe.abbr" has changed from "+11" to "+1030".
The field "Australia/Lord_Howe.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Melbourne.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Melbourne.offset" has changed from "+11:00" to "+10:00".
The field "Australia/NSW.abbr" has changed from "AEDT" to "AEST".
The field "Australia/NSW.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Sydney.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Sydney.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Tasmania.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Tasmania.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Victoria.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Victoria.offset" has changed from "+11:00" to "+10:00".
The field "Pacific/Norfolk.abbr" has changed from "+12" to "+11".
The field "Pacific/Norfolk.offset" has changed from "+12:00" to "+11:00".
The field "Antarctica/McMurdo.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/McMurdo.offset" has changed from "+13:00" to "+12:00".
The field "Antarctica/South_Pole.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/South_Pole.offset" has changed from "+13:00" to "+12:00".
The field "NZ.abbr" has changed from "NZDT" to "NZST".
The field "NZ.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Auckland.abbr" has changed from "NZDT" to "NZST".
The field "Pacific/Auckland.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Fiji.abbr" has changed from "+13" to "+12".
The field "Pacific/Fiji.offset" has changed from "+13:00" to "+12:00".
The field "NZ-CHAT.abbr" has changed from "+1345" to "+1245".
The field "NZ-CHAT.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Chatham.abbr" has changed from "+1345" to "+1245".
The field "Pacific/Chatham.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Apia.abbr" has changed from "+14" to "+13".
The field "Pacific/Apia.offset" has changed from "+14:00" to "+13:00".
The field "America/Nuuk" has been added ('{"name":"America/Nuuk","abbr":"-02","offset":"-02:00"}').


[timezone] Data generated with warnings for locale "ja".

The field "America/Adak.abbr" has changed from "HST" to "HDT".
The field "America/Adak.offset" has changed from "-10:00" to "-09:00".
The field "America/Atka.abbr" has changed from "HST" to "HDT".
The field "America/Atka.offset" has changed from "-10:00" to "-09:00".
The field "US/Aleutian.abbr" has changed from "HST" to "HDT".
The field "US/Aleutian.offset" has changed from "-10:00" to "-09:00".
The field "America/Anchorage.abbr" has changed from "AKST" to "AKDT".
The field "America/Anchorage.offset" has changed from "-09:00" to "-08:00".
The field "America/Juneau.abbr" has changed from "AKST" to "AKDT".
The field "America/Juneau.offset" has changed from "-09:00" to "-08:00".
The field "America/Metlakatla.abbr" has changed from "AKST" to "AKDT".
The field "America/Metlakatla.offset" has changed from "-09:00" to "-08:00".
The field "America/Nome.abbr" has changed from "AKST" to "AKDT".
The field "America/Nome.offset" has changed from "-09:00" to "-08:00".
The field "America/Sitka.abbr" has changed from "AKST" to "AKDT".
The field "America/Sitka.offset" has changed from "-09:00" to "-08:00".
The field "America/Yakutat.abbr" has changed from "AKST" to "AKDT".
The field "America/Yakutat.offset" has changed from "-09:00" to "-08:00".
The field "US/Alaska.abbr" has changed from "AKST" to "AKDT".
The field "US/Alaska.offset" has changed from "-09:00" to "-08:00".
The field "America/Dawson.abbr" has changed from "PST" to "MST".
The field "America/Dawson.offset" has changed from "-08:00" to "-07:00".
The field "America/Ensenada.abbr" has changed from "PST" to "PDT".
The field "America/Ensenada.offset" has changed from "-08:00" to "-07:00".
The field "America/Los_Angeles.abbr" has changed from "PST" to "PDT".
The field "America/Los_Angeles.offset" has changed from "-08:00" to "-07:00".
The field "America/Santa_Isabel.abbr" has changed from "PST" to "PDT".
The field "America/Santa_Isabel.offset" has changed from "-08:00" to "-07:00".
The field "America/Tijuana.abbr" has changed from "PST" to "PDT".
The field "America/Tijuana.offset" has changed from "-08:00" to "-07:00".
The field "America/Vancouver.abbr" has changed from "PST" to "PDT".
The field "America/Vancouver.offset" has changed from "-08:00" to "-07:00".
The field "America/Whitehorse.abbr" has changed from "PST" to "MST".
The field "America/Whitehorse.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Pacific.abbr" has changed from "PST" to "PDT".
The field "Canada/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "Canada/Yukon.abbr" has changed from "PST" to "MST".
The field "Canada/Yukon.offset" has changed from "-08:00" to "-07:00".
The field "Mexico/BajaNorte.abbr" has changed from "PST" to "PDT".
The field "Mexico/BajaNorte.offset" has changed from "-08:00" to "-07:00".
The field "PST8PDT.abbr" has changed from "PST" to "PDT".
The field "PST8PDT.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific.abbr" has changed from "PST" to "PDT".
The field "US/Pacific.offset" has changed from "-08:00" to "-07:00".
The field "US/Pacific-New.abbr" has changed from "PST" to "PDT".
The field "US/Pacific-New.offset" has changed from "-08:00" to "-07:00".
The field "America/Boise.abbr" has changed from "MST" to "MDT".
The field "America/Boise.offset" has changed from "-07:00" to "-06:00".
The field "America/Cambridge_Bay.abbr" has changed from "MST" to "MDT".
The field "America/Cambridge_Bay.offset" has changed from "-07:00" to "-06:00".
The field "America/Chihuahua.abbr" has changed from "MST" to "MDT".
The field "America/Chihuahua.offset" has changed from "-07:00" to "-06:00".
The field "America/Denver.abbr" has changed from "MST" to "MDT".
The field "America/Denver.offset" has changed from "-07:00" to "-06:00".
The field "America/Edmonton.abbr" has changed from "MST" to "MDT".
The field "America/Edmonton.offset" has changed from "-07:00" to "-06:00".
The field "America/Inuvik.abbr" has changed from "MST" to "MDT".
The field "America/Inuvik.offset" has changed from "-07:00" to "-06:00".
The field "America/Mazatlan.abbr" has changed from "MST" to "MDT".
The field "America/Mazatlan.offset" has changed from "-07:00" to "-06:00".
The field "America/Ojinaga.abbr" has changed from "MST" to "MDT".
The field "America/Ojinaga.offset" has changed from "-07:00" to "-06:00".
The field "America/Shiprock.abbr" has changed from "MST" to "MDT".
The field "America/Shiprock.offset" has changed from "-07:00" to "-06:00".
The field "America/Yellowknife.abbr" has changed from "MST" to "MDT".
The field "America/Yellowknife.offset" has changed from "-07:00" to "-06:00".
The field "Canada/Mountain.abbr" has changed from "MST" to "MDT".
The field "Canada/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "MST7MDT.abbr" has changed from "MST" to "MDT".
The field "MST7MDT.offset" has changed from "-07:00" to "-06:00".
The field "Mexico/BajaSur.abbr" has changed from "MST" to "MDT".
The field "Mexico/BajaSur.offset" has changed from "-07:00" to "-06:00".
The field "Navajo.abbr" has changed from "MST" to "MDT".
The field "Navajo.offset" has changed from "-07:00" to "-06:00".
The field "US/Mountain.abbr" has changed from "MST" to "MDT".
The field "US/Mountain.offset" has changed from "-07:00" to "-06:00".
The field "America/Bahia_Banderas.abbr" has changed from "CST" to "CDT".
The field "America/Bahia_Banderas.offset" has changed from "-06:00" to "-05:00".
The field "America/Chicago.abbr" has changed from "CST" to "CDT".
The field "America/Chicago.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Knox.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Knox.offset" has changed from "-06:00" to "-05:00".
The field "America/Indiana/Tell_City.abbr" has changed from "CST" to "CDT".
The field "America/Indiana/Tell_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Knox_IN.abbr" has changed from "CST" to "CDT".
The field "America/Knox_IN.offset" has changed from "-06:00" to "-05:00".
The field "America/Matamoros.abbr" has changed from "CST" to "CDT".
The field "America/Matamoros.offset" has changed from "-06:00" to "-05:00".
The field "America/Menominee.abbr" has changed from "CST" to "CDT".
The field "America/Menominee.offset" has changed from "-06:00" to "-05:00".
The field "America/Merida.abbr" has changed from "CST" to "CDT".
The field "America/Merida.offset" has changed from "-06:00" to "-05:00".
The field "America/Mexico_City.abbr" has changed from "CST" to "CDT".
The field "America/Mexico_City.offset" has changed from "-06:00" to "-05:00".
The field "America/Monterrey.abbr" has changed from "CST" to "CDT".
The field "America/Monterrey.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Beulah.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Beulah.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/Center.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/Center.offset" has changed from "-06:00" to "-05:00".
The field "America/North_Dakota/New_Salem.abbr" has changed from "CST" to "CDT".
The field "America/North_Dakota/New_Salem.offset" has changed from "-06:00" to "-05:00".
The field "America/Rainy_River.abbr" has changed from "CST" to "CDT".
The field "America/Rainy_River.offset" has changed from "-06:00" to "-05:00".
The field "America/Rankin_Inlet.abbr" has changed from "CST" to "CDT".
The field "America/Rankin_Inlet.offset" has changed from "-06:00" to "-05:00".
The field "America/Resolute.abbr" has changed from "CST" to "CDT".
The field "America/Resolute.offset" has changed from "-06:00" to "-05:00".
The field "America/Winnipeg.abbr" has changed from "CST" to "CDT".
The field "America/Winnipeg.offset" has changed from "-06:00" to "-05:00".
The field "CST6CDT.abbr" has changed from "CST" to "CDT".
The field "CST6CDT.offset" has changed from "-06:00" to "-05:00".
The field "Canada/Central.abbr" has changed from "CST" to "CDT".
The field "Canada/Central.offset" has changed from "-06:00" to "-05:00".
The field "Mexico/General.abbr" has changed from "CST" to "CDT".
The field "Mexico/General.offset" has changed from "-06:00" to "-05:00".
The field "US/Central.abbr" has changed from "CST" to "CDT".
The field "US/Central.offset" has changed from "-06:00" to "-05:00".
The field "US/Indiana-Starke.abbr" has changed from "CST" to "CDT".
The field "US/Indiana-Starke.offset" has changed from "-06:00" to "-05:00".
The field "America/Detroit.abbr" has changed from "EST" to "EDT".
The field "America/Detroit.offset" has changed from "-05:00" to "-04:00".
The field "America/Fort_Wayne.abbr" has changed from "EST" to "EDT".
The field "America/Fort_Wayne.offset" has changed from "-05:00" to "-04:00".
The field "America/Grand_Turk.abbr" has changed from "EST" to "EDT".
The field "America/Grand_Turk.offset" has changed from "-05:00" to "-04:00".
The field "America/Havana.abbr" has changed from "CST" to "CDT".
The field "America/Havana.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Marengo.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Marengo.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Petersburg.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Petersburg.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vevay.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vevay.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Vincennes.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Vincennes.offset" has changed from "-05:00" to "-04:00".
The field "America/Indiana/Winamac.abbr" has changed from "EST" to "EDT".
The field "America/Indiana/Winamac.offset" has changed from "-05:00" to "-04:00".
The field "America/Indianapolis.abbr" has changed from "EST" to "EDT".
The field "America/Indianapolis.offset" has changed from "-05:00" to "-04:00".
The field "America/Iqaluit.abbr" has changed from "EST" to "EDT".
The field "America/Iqaluit.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Kentucky/Monticello.abbr" has changed from "EST" to "EDT".
The field "America/Kentucky/Monticello.offset" has changed from "-05:00" to "-04:00".
The field "America/Louisville.abbr" has changed from "EST" to "EDT".
The field "America/Louisville.offset" has changed from "-05:00" to "-04:00".
The field "America/Montreal.abbr" has changed from "EST" to "EDT".
The field "America/Montreal.offset" has changed from "-05:00" to "-04:00".
The field "America/Nassau.abbr" has changed from "EST" to "EDT".
The field "America/Nassau.offset" has changed from "-05:00" to "-04:00".
The field "America/New_York.abbr" has changed from "EST" to "EDT".
The field "America/New_York.offset" has changed from "-05:00" to "-04:00".
The field "America/Nipigon.abbr" has changed from "EST" to "EDT".
The field "America/Nipigon.offset" has changed from "-05:00" to "-04:00".
The field "America/Pangnirtung.abbr" has changed from "EST" to "EDT".
The field "America/Pangnirtung.offset" has changed from "-05:00" to "-04:00".
The field "America/Port-au-Prince.abbr" has changed from "EST" to "EDT".
The field "America/Port-au-Prince.offset" has changed from "-05:00" to "-04:00".
The field "America/Thunder_Bay.abbr" has changed from "EST" to "EDT".
The field "America/Thunder_Bay.offset" has changed from "-05:00" to "-04:00".
The field "America/Toronto.abbr" has changed from "EST" to "EDT".
The field "America/Toronto.offset" has changed from "-05:00" to "-04:00".
The field "Canada/Eastern.abbr" has changed from "EST" to "EDT".
The field "Canada/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "Chile/EasterIsland.abbr" has changed from "-05" to "-06".
The field "Chile/EasterIsland.offset" has changed from "-05:00" to "-06:00".
The field "Cuba.abbr" has changed from "CST" to "CDT".
The field "Cuba.offset" has changed from "-05:00" to "-04:00".
The field "EST5EDT.abbr" has changed from "EST" to "EDT".
The field "EST5EDT.offset" has changed from "-05:00" to "-04:00".
The field "Pacific/Easter.abbr" has changed from "-05" to "-06".
The field "Pacific/Easter.offset" has changed from "-05:00" to "-06:00".
The field "US/East-Indiana.abbr" has changed from "EST" to "EDT".
The field "US/East-Indiana.offset" has changed from "-05:00" to "-04:00".
The field "US/Eastern.abbr" has changed from "EST" to "EDT".
The field "US/Eastern.offset" has changed from "-05:00" to "-04:00".
The field "US/Michigan.abbr" has changed from "EST" to "EDT".
The field "US/Michigan.offset" has changed from "-05:00" to "-04:00".
The field "America/Glace_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Glace_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Goose_Bay.abbr" has changed from "AST" to "ADT".
The field "America/Goose_Bay.offset" has changed from "-04:00" to "-03:00".
The field "America/Halifax.abbr" has changed from "AST" to "ADT".
The field "America/Halifax.offset" has changed from "-04:00" to "-03:00".
The field "America/Moncton.abbr" has changed from "AST" to "ADT".
The field "America/Moncton.offset" has changed from "-04:00" to "-03:00".
The field "America/Thule.abbr" has changed from "AST" to "ADT".
The field "America/Thule.offset" has changed from "-04:00" to "-03:00".
The field "Atlantic/Bermuda.abbr" has changed from "AST" to "ADT".
The field "Atlantic/Bermuda.offset" has changed from "-04:00" to "-03:00".
The field "Canada/Atlantic.abbr" has changed from "AST" to "ADT".
The field "Canada/Atlantic.offset" has changed from "-04:00" to "-03:00".
The field "America/St_Johns.abbr" has changed from "NST" to "NDT".
The field "America/St_Johns.offset" has changed from "-03:30" to "-02:30".
The field "Canada/Newfoundland.abbr" has changed from "NST" to "NDT".
The field "Canada/Newfoundland.offset" has changed from "-03:30" to "-02:30".
The field "America/Asuncion.abbr" has changed from "-03" to "-04".
The field "America/Asuncion.offset" has changed from "-03:00" to "-04:00".
The field "America/Godthab.abbr" has changed from "-03" to "-02".
The field "America/Godthab.offset" has changed from "-03:00" to "-02:00".
The field "America/Miquelon.abbr" has changed from "-03" to "-02".
The field "America/Miquelon.offset" has changed from "-03:00" to "-02:00".
The field "America/Santiago.abbr" has changed from "-03" to "-04".
The field "America/Santiago.offset" has changed from "-03:00" to "-04:00".
The field "Chile/Continental.abbr" has changed from "-03" to "-04".
The field "Chile/Continental.offset" has changed from "-03:00" to "-04:00".
The field "America/Scoresbysund.abbr" has changed from "-01" to "+00".
The field "America/Scoresbysund.offset" has changed from "-01:00" to "+00:00".
The field "Atlantic/Azores.abbr" has changed from "-01" to "+00".
The field "Atlantic/Azores.offset" has changed from "-01:00" to "+00:00".
The field "Antarctica/Troll.abbr" has changed from "+00" to "+02".
The field "Antarctica/Troll.offset" has changed from "+00:00" to "+02:00".
The field "Atlantic/Canary.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Canary.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faeroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faeroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Faroe.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Faroe.offset" has changed from "+00:00" to "+01:00".
The field "Atlantic/Madeira.abbr" has changed from "WET" to "WEST".
The field "Atlantic/Madeira.offset" has changed from "+00:00" to "+01:00".
The field "Eire.abbr" has changed from "GMT" to "IST".
The field "Eire.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Belfast.abbr" has changed from "GMT" to "BST".
The field "Europe/Belfast.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Dublin.abbr" has changed from "GMT" to "IST".
The field "Europe/Dublin.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Guernsey.abbr" has changed from "GMT" to "BST".
The field "Europe/Guernsey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Isle_of_Man.abbr" has changed from "GMT" to "BST".
The field "Europe/Isle_of_Man.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Jersey.abbr" has changed from "GMT" to "BST".
The field "Europe/Jersey.offset" has changed from "+00:00" to "+01:00".
The field "Europe/Lisbon.abbr" has changed from "WET" to "WEST".
The field "Europe/Lisbon.offset" has changed from "+00:00" to "+01:00".
The field "Europe/London.abbr" has changed from "GMT" to "BST".
The field "Europe/London.offset" has changed from "+00:00" to "+01:00".
The field "GB.abbr" has changed from "GMT" to "BST".
The field "GB.offset" has changed from "+00:00" to "+01:00".
The field "GB-Eire.abbr" has changed from "GMT" to "BST".
The field "GB-Eire.offset" has changed from "+00:00" to "+01:00".
The field "Portugal.abbr" has changed from "WET" to "WEST".
The field "Portugal.offset" has changed from "+00:00" to "+01:00".
The field "WET.abbr" has changed from "WET" to "WEST".
The field "WET.offset" has changed from "+00:00" to "+01:00".
The field "Africa/Ceuta.abbr" has changed from "CET" to "CEST".
The field "Africa/Ceuta.offset" has changed from "+01:00" to "+02:00".
The field "Arctic/Longyearbyen.abbr" has changed from "CET" to "CEST".
The field "Arctic/Longyearbyen.offset" has changed from "+01:00" to "+02:00".
The field "Atlantic/Jan_Mayen.abbr" has changed from "CET" to "CEST".
The field "Atlantic/Jan_Mayen.offset" has changed from "+01:00" to "+02:00".
The field "CET.abbr" has changed from "CET" to "CEST".
The field "CET.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Amsterdam.abbr" has changed from "CET" to "CEST".
The field "Europe/Amsterdam.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Andorra.abbr" has changed from "CET" to "CEST".
The field "Europe/Andorra.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Belgrade.abbr" has changed from "CET" to "CEST".
The field "Europe/Belgrade.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Berlin.abbr" has changed from "CET" to "CEST".
The field "Europe/Berlin.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Bratislava.abbr" has changed from "CET" to "CEST".
The field "Europe/Bratislava.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Brussels.abbr" has changed from "CET" to "CEST".
The field "Europe/Brussels.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Budapest.abbr" has changed from "CET" to "CEST".
The field "Europe/Budapest.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Busingen.abbr" has changed from "CET" to "CEST".
The field "Europe/Busingen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Copenhagen.abbr" has changed from "CET" to "CEST".
The field "Europe/Copenhagen.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Gibraltar.abbr" has changed from "CET" to "CEST".
The field "Europe/Gibraltar.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Ljubljana.abbr" has changed from "CET" to "CEST".
The field "Europe/Ljubljana.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Luxembourg.abbr" has changed from "CET" to "CEST".
The field "Europe/Luxembourg.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Madrid.abbr" has changed from "CET" to "CEST".
The field "Europe/Madrid.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Malta.abbr" has changed from "CET" to "CEST".
The field "Europe/Malta.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Monaco.abbr" has changed from "CET" to "CEST".
The field "Europe/Monaco.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Oslo.abbr" has changed from "CET" to "CEST".
The field "Europe/Oslo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Paris.abbr" has changed from "CET" to "CEST".
The field "Europe/Paris.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Podgorica.abbr" has changed from "CET" to "CEST".
The field "Europe/Podgorica.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Prague.abbr" has changed from "CET" to "CEST".
The field "Europe/Prague.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Rome.abbr" has changed from "CET" to "CEST".
The field "Europe/Rome.offset" has changed from "+01:00" to "+02:00".
The field "Europe/San_Marino.abbr" has changed from "CET" to "CEST".
The field "Europe/San_Marino.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Sarajevo.abbr" has changed from "CET" to "CEST".
The field "Europe/Sarajevo.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Skopje.abbr" has changed from "CET" to "CEST".
The field "Europe/Skopje.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Stockholm.abbr" has changed from "CET" to "CEST".
The field "Europe/Stockholm.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Tirane.abbr" has changed from "CET" to "CEST".
The field "Europe/Tirane.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vaduz.abbr" has changed from "CET" to "CEST".
The field "Europe/Vaduz.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vatican.abbr" has changed from "CET" to "CEST".
The field "Europe/Vatican.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Vienna.abbr" has changed from "CET" to "CEST".
The field "Europe/Vienna.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Warsaw.abbr" has changed from "CET" to "CEST".
The field "Europe/Warsaw.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zagreb.abbr" has changed from "CET" to "CEST".
The field "Europe/Zagreb.offset" has changed from "+01:00" to "+02:00".
The field "Europe/Zurich.abbr" has changed from "CET" to "CEST".
The field "Europe/Zurich.offset" has changed from "+01:00" to "+02:00".
The field "MET.abbr" has changed from "MET" to "MEST".
The field "MET.offset" has changed from "+01:00" to "+02:00".
The field "Poland.abbr" has changed from "CET" to "CEST".
The field "Poland.offset" has changed from "+01:00" to "+02:00".
The field "Asia/Amman.abbr" has changed from "EET" to "EEST".
The field "Asia/Amman.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Beirut.abbr" has changed from "EET" to "EEST".
The field "Asia/Beirut.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Damascus.abbr" has changed from "EET" to "EEST".
The field "Asia/Damascus.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Famagusta.abbr" has changed from "EET" to "EEST".
The field "Asia/Famagusta.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Gaza.abbr" has changed from "EET" to "EEST".
The field "Asia/Gaza.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Hebron.abbr" has changed from "EET" to "EEST".
The field "Asia/Hebron.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Jerusalem.abbr" has changed from "IST" to "IDT".
The field "Asia/Jerusalem.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Asia/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tel_Aviv.abbr" has changed from "IST" to "IDT".
The field "Asia/Tel_Aviv.offset" has changed from "+02:00" to "+03:00".
The field "EET.abbr" has changed from "EET" to "EEST".
The field "EET.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Athens.abbr" has changed from "EET" to "EEST".
The field "Europe/Athens.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Bucharest.abbr" has changed from "EET" to "EEST".
The field "Europe/Bucharest.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Chisinau.abbr" has changed from "EET" to "EEST".
The field "Europe/Chisinau.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Helsinki.abbr" has changed from "EET" to "EEST".
The field "Europe/Helsinki.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Kiev.abbr" has changed from "EET" to "EEST".
The field "Europe/Kiev.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Mariehamn.abbr" has changed from "EET" to "EEST".
The field "Europe/Mariehamn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Nicosia.abbr" has changed from "EET" to "EEST".
The field "Europe/Nicosia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Riga.abbr" has changed from "EET" to "EEST".
The field "Europe/Riga.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Sofia.abbr" has changed from "EET" to "EEST".
The field "Europe/Sofia.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tallinn.abbr" has changed from "EET" to "EEST".
The field "Europe/Tallinn.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Tiraspol.abbr" has changed from "EET" to "EEST".
The field "Europe/Tiraspol.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Uzhgorod.abbr" has changed from "EET" to "EEST".
The field "Europe/Uzhgorod.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Vilnius.abbr" has changed from "EET" to "EEST".
The field "Europe/Vilnius.offset" has changed from "+02:00" to "+03:00".
The field "Europe/Zaporozhye.abbr" has changed from "EET" to "EEST".
The field "Europe/Zaporozhye.offset" has changed from "+02:00" to "+03:00".
The field "Israel.abbr" has changed from "IST" to "IDT".
The field "Israel.offset" has changed from "+02:00" to "+03:00".
The field "Asia/Tehran.abbr" has changed from "+0330" to "+0430".
The field "Asia/Tehran.offset" has changed from "+03:30" to "+04:30".
The field "Iran.abbr" has changed from "+0330" to "+0430".
The field "Iran.offset" has changed from "+03:30" to "+04:30".
The field "Australia/Adelaide.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Adelaide.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Broken_Hill.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Broken_Hill.offset" has changed from "+10:30" to "+09:30".
The field "Australia/South.abbr" has changed from "ACDT" to "ACST".
The field "Australia/South.offset" has changed from "+10:30" to "+09:30".
The field "Australia/Yancowinna.abbr" has changed from "ACDT" to "ACST".
The field "Australia/Yancowinna.offset" has changed from "+10:30" to "+09:30".
The field "Australia/ACT.abbr" has changed from "AEDT" to "AEST".
The field "Australia/ACT.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Canberra.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Canberra.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Currie.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Currie.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Hobart.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Hobart.offset" has changed from "+11:00" to "+10:00".
The field "Australia/LHI.abbr" has changed from "+11" to "+1030".
The field "Australia/LHI.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Lord_Howe.abbr" has changed from "+11" to "+1030".
The field "Australia/Lord_Howe.offset" has changed from "+11:00" to "+10:30".
The field "Australia/Melbourne.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Melbourne.offset" has changed from "+11:00" to "+10:00".
The field "Australia/NSW.abbr" has changed from "AEDT" to "AEST".
The field "Australia/NSW.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Sydney.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Sydney.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Tasmania.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Tasmania.offset" has changed from "+11:00" to "+10:00".
The field "Australia/Victoria.abbr" has changed from "AEDT" to "AEST".
The field "Australia/Victoria.offset" has changed from "+11:00" to "+10:00".
The field "Pacific/Norfolk.abbr" has changed from "+12" to "+11".
The field "Pacific/Norfolk.offset" has changed from "+12:00" to "+11:00".
The field "Antarctica/McMurdo.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/McMurdo.offset" has changed from "+13:00" to "+12:00".
The field "Antarctica/South_Pole.abbr" has changed from "NZDT" to "NZST".
The field "Antarctica/South_Pole.offset" has changed from "+13:00" to "+12:00".
The field "NZ.abbr" has changed from "NZDT" to "NZST".
The field "NZ.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Auckland.abbr" has changed from "NZDT" to "NZST".
The field "Pacific/Auckland.offset" has changed from "+13:00" to "+12:00".
The field "Pacific/Fiji.abbr" has changed from "+13" to "+12".
The field "Pacific/Fiji.offset" has changed from "+13:00" to "+12:00".
The field "NZ-CHAT.abbr" has changed from "+1345" to "+1245".
The field "NZ-CHAT.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Chatham.abbr" has changed from "+1345" to "+1245".
The field "Pacific/Chatham.offset" has changed from "+13:45" to "+12:45".
The field "Pacific/Apia.abbr" has changed from "+14" to "+13".
The field "Pacific/Apia.offset" has changed from "+14:00" to "+13:00".
The field "America/Nuuk" has been added ('{"name":"America/Nuuk","abbr":"-02","offset":"-02:00"}').
[language] Updated 6 locales


[language] Data generated with warnings for locale "de".

The field "en-SZ.country" has changed from "Swasiland" to "Eswatini".
The field "ss-SZ.country" has changed from "Swasiland" to "Eswatini".


[language] Data generated with warnings for locale "es".

The field "mk-MK.country" has changed from "Macedonia" to "Macedonia del Norte".
The field "pa.language" has changed from "panyabí" to "punyabí".


[language] Data generated with warnings for locale "fr-FR".

The field "mk-MK.country" has changed from "Macédoine" to "Macédoine du Nord".
[currency] Writing en data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/currencies/en.json
[currency] Writing ja data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/currencies/ja.json
[currency] Writing zh-CN data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/currencies/zh-CN.json
[currency] Writing es data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/currencies/es.json
[currency] Writing fr-FR data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/currencies/fr-FR.json
[currency] Writing de data to /Users/emmenko/dev/src/merchant-center-application-kit/packages/l10n/data/currencies/de.json
[currency] Updated 6 locales


[currency] Data generated with warnings for locale "es".

The field "MWK.label" has changed from "kwacha malauí" to "kuacha malauí".
The field "ZMW.label" has changed from "kwacha zambiano" to "kuacha zambiano".
```

</details>

Reply:
Makes sense to me what I can see but... It's always a bit hard to know for sure.